<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>2025.knit</title>

<script src="site_libs/header-attrs-2.22/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Research Progress</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Progress
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="2025.html">2025</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/zq2209/GradPLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-02-03
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>GradPLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220610code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220610code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220610)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtreeeffd6aa650b2cea68f2b60211c07d6e9814cdb7atargetblankeffd6aaa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/zq2209/GradPLog/tree/effd6aa650b2cea68f2b60211c07d6e9814cdb7a" target="_blank">effd6aa</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtreeeffd6aa650b2cea68f2b60211c07d6e9814cdb7atargetblankeffd6aaa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/zq2209/GradPLog/tree/effd6aa650b2cea68f2b60211c07d6e9814cdb7a" target="_blank">effd6aa</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/2025.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/2025.Rmd</code>) and HTML
(<code>docs/2025.html</code>) files. If you’ve configured a remote Git
repository (see <code>?wflow_git_remote</code>), click on the hyperlinks
in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/effd6aa650b2cea68f2b60211c07d6e9814cdb7a/analysis/2025.Rmd" target="_blank">effd6aa</a>
</td>
<td>
“John
</td>
<td>
2025-02-01
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/390b75e3d99f0c1a4e6b25cd664445772f998efa/analysis/2025.Rmd" target="_blank">390b75e</a>
</td>
<td>
“John
</td>
<td>
2025-01-31
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/390b75e3d99f0c1a4e6b25cd664445772f998efa/docs/2025.html" target="_blank">390b75e</a>
</td>
<td>
“John
</td>
<td>
2025-01-31
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="current-research-on-ibd" class="section level2">
<h2>Current Research on IBD</h2>
<p>This section summarize the WES and GWAS studies on IBD, giving
background information on the disease and current research on the
diease.</p>
<div id="disease-background" class="section level3">
<h3>Disease Background</h3>
<div id="general-background-of-ibd" class="section level4">
<h4>General Background of IBD</h4>
<p><img src="assets/image1.png" style="width:60.0%" /></p>
<ul>
<li><p>IBD is Chronic inflammation influenced by genetics, environment,
microbiota, and immunity.</p></li>
<li><p>Genetic Contribution</p>
<ul>
<li><p>Crohn’s Disease (CD): 15% family history; twin studies show 50%
concordance in monozygotic (MZ) twins vs. less than 10% in dizygotic
(DZ) twins.</p></li>
<li><p>GWAS identified 163 loci; trans-ethnic studies identified an
additional 38 loci.</p></li>
</ul></li>
<li><p>Epigenetics</p>
<ul>
<li><p>Genome-environment interactions affect disease
progression.</p></li>
<li><p>Emerging research focuses on the role of epigenetics in
IBD.</p></li>
</ul></li>
</ul>
</div>
<div id="genetic-associations-in-ibd" class="section level4">
<h4>Genetic Associations in IBD</h4>
<ul>
<li><p>NOD2: First CD-associated gene (2001), with key variants R702W
and G908R.</p></li>
<li><p>Autophagy Genes: ATG16L1, LRRK2, and IRGM predispose individuals
to IBD.</p></li>
<li><p>IL-10 Receptor Mutations: IL10RA, IL10RB are linked to
colitis.</p></li>
<li><p>IBD-Associated Loci: ~240 loci identified (as of 2022); 30 shared
between CD and UC.</p></li>
<li><p>CD Predictive Loci: FOXO3, IGFBP1, and XACT as potential
markers.</p></li>
</ul>
</div>
</div>
<div
id="large-scale-exome-sequencing-identifies-novel-genes-and-pathways-in-crohns-disease"
class="section level3">
<h3>Large-Scale Exome Sequencing Identifies Novel Genes and Pathways in
Crohn’s Disease</h3>
<p>Sazonovs, Aleksejs et al.</p>
<div id="motivation" class="section level4">
<h4>Motivation</h4>
<ul>
<li><p>CD is a chronic inflammatory disorder with a strong genetic
component.</p></li>
<li><p>GWAS has primarily focused on common variants, but rare coding
variants remain under-explored.</p></li>
</ul>
</div>
<div id="study-design" class="section level4">
<h4>Study Design</h4>
<p><img src="assets/image2.png" style="width:60.0%" /></p>
</div>
<div id="findings" class="section level4">
<h4>Findings</h4>
<ul>
<li>Single Variant Analysis: 94 out of 116 variants replicated in
discovery datasets.</li>
</ul>
<p><img src="assets/image3.png" style="width:80.0%" /> <img
src="assets/image4.png" style="width:60.0%" /></p>
<ul>
<li><p>Burden Test: Missense vs. frameshift variants analyzed. <img
src="assets/image5.png" style="width:80.0%" /> #### Takeaways</p></li>
<li><p>Exome Sequencing Complements GWAS</p>
<ul>
<li>Addresses gaps in genetic architecture (low-frequency and rare
variants) that earlier CD GWAS meta-analyses could not observe.</li>
</ul></li>
<li><p>Significance of Coding Variants:</p>
<ul>
<li>Coding variants, although fewer than noncoding variants, are highly
enriched for associations with both common and rare diseases.</li>
<li>Tend to have stronger effects due to natural selection, which often
keeps their frequencies low.</li>
<li>Provide direct links to specific genes and pathogenic mechanisms,
unlike noncoding variants.</li>
</ul></li>
<li><p>Key Findings in CD Pathogenesis:</p>
<ul>
<li>Novel genes such as PDLIM5, SDF2L1, HGFAC, PAF-R, and CCR7 emphasize
the role of mesenchymal cells (MCs) in intestinal inflammation.</li>
</ul></li>
<li><p>Therapeutic Implications:</p>
<ul>
<li>Findings highlight the potential for therapeutic strategies
targeting mesenchymal niche balance to address CD pathogenesis.</li>
<li>Genetic evidence for drug targets is valuable for driving drug
development.</li>
</ul></li>
<li><p>Future Directions:</p>
<ul>
<li>Expanded sequencing efforts in ulcerative colitis and integrated
analyses with larger GWAS studies are expected to identify more linked
genes.</li>
</ul></li>
</ul>
</div>
</div>
<div
id="genome-wide-association-study-implicates-immune-activation-of-multiple-integrin-genes-in-inflammatory-bowel-disease"
class="section level3">
<h3>Genome-wide association study implicates immune activation of
multiple integrin genes in inflammatory bowel disease</h3>
<p>de Lange, K., Moutsianas, L., Lee, J. et al.</p>
<div id="motivation-1" class="section level4">
<h4>Motivation</h4>
<ul>
<li><p>Current treatments involve immunomodulators, but patients often
experience side effects or treatment resistance.</p></li>
<li><p>GWAS and Immunochip studies have identified risk loci but have
had limited therapeutic impact.</p></li>
</ul>
</div>
<div id="study-design-1" class="section level4">
<h4>Study Design</h4>
<p><img src="assets/image5.png" style="width:90.0%" /></p>
</div>
<div id="findings-1" class="section level4">
<h4>Findings</h4>
<p>Identified 25 new GWAS loci</p>
<ul>
<li>4 loci with significant variants
<ul>
<li>SLAMF8</li>
<li>RORC</li>
<li>PLCG2</li>
<li>NCF4</li>
</ul></li>
<li>Another 4 loci within integrin gene
<ul>
<li>3 loci showed larger than 90% probability have colocalization with
eQTL</li>
<li>1 loci showed intermediate evidendence</li>
</ul></li>
</ul>
</div>
<div id="takeaways" class="section level4">
<h4>Takeaways</h4>
<ul>
<li><p>Integrins are not only important in cell trafficking but can also
participate in cellular signaling.</p></li>
<li><p>Highlighted integrins as key therapeutic targets:</p>
<ul>
<li>Monoclonal antibodies like vedolizumab and etrolizumab targeting
integrins have shown efficacy in IBD.</li>
<li>Identified SMAD7, a modulator of <span
class="math inline">\(TGF-\beta\)</span> signaling, as a potential
target for Crohn’s disease treatment.</li>
</ul></li>
<li><p>Emphasized the importance of gut-selective therapies to minimize
risks like progressive multifocal leukoencephalopathy (PML).</p></li>
<li><p>These discoveries have demonstrated that the effect sizes of GWAS
associations do not necessarily reflect the importance or therapeutic
relevance of their underlying biological pathways.</p></li>
</ul>
</div>
</div>
</div>
<div id="method" class="section level2">
<h2>Method</h2>
<div
id="prioritizing-disease-mediating-genes-leveraging-trans-gene-regulation"
class="section level3">
<h3>Prioritizing disease-mediating genes leveraging trans-gene
regulation</h3>
<div id="motivation-2" class="section level4">
<h4>Motivation</h4>
<ul>
<li>Complex Genetic Architecture of Diseases:
<ul>
<li>Complex traits are highly polygenic, involving numerous genes with
varying effect sizes.</li>
<li>The omnigenic model posits that all expressed genes in
disease-relevant tissues contribute to disease risk, but not all genes
have equal importance.</li>
</ul></li>
<li>Importance of Disease-Mediating Genes:
<ul>
<li>Disease-associated genes can be categorized into:
<ul>
<li>Disease-mediating genes: Directly affect the disease.</li>
<li>Peripheral genes: Indirectly influence the disease through
trans-regulatory networks.</li>
</ul></li>
<li>Identifying disease-mediating genes is critical for uncovering
pathways and mechanisms relevant to treatment development.</li>
</ul></li>
<li>Limitations of Current Gene Prioritization Methods:
<ul>
<li>GWAS:
<ul>
<li>Identifies important disease-mediating genes but may miss many due
to lack of power in detecting low-frequency or rare variants with large
effects.</li>
</ul></li>
<li>Rare Variant Methods:
<ul>
<li>Gene-based approaches, like burden tests in WES, improve detection
but are limited by current sample sizes.</li>
</ul></li>
<li>Graph and Canonical Correlation Models:
<ul>
<li>Recent approaches predict candidate genes or link expressions to
variants but fail to explicitly differentiate disease-mediating from
peripheral genes or reveal trans-regulatory networks.</li>
</ul></li>
</ul></li>
<li>Challenges with Cis-eQTLs in Disease Gene Discovery:
<ul>
<li>Disease genes often lack strong cis-QTL signals due to stronger
selective constraints.</li>
<li>Cis-eQTL limitations necessitate focusing on trans-QTLs, as their
target genes share similar selective constraints with disease
genes.</li>
</ul></li>
<li>Significance of Trans-Regulation:
<ul>
<li>Trans-regulatory networks explain a large portion (~70%) of disease
heritability.</li>
<li>Understanding trans-regulation provides insights into collaborative
gene contributions to disease risk and highlights the importance of
completing trans-gene regulatory networks for understanding complex
trait genetics.</li>
</ul></li>
</ul>
</div>
<div id="cota" class="section level4">
<h4>COTA</h4>
<ul>
<li><p>COTA Model: Integrates trans-regulatory effects to identify
disease-mediating genes.</p></li>
<li><p>GBAT: Predicts gene expression using machine learning
models.</p></li>
</ul>
<p><img src="assets/image7.png" style="width:60.0%" /></p>
<ul>
<li>DACT: Estimates gene effects from burden tests.</li>
</ul>
<p><img src="assets/image8.png" style="width:60.0%" /></p>
</div>
<div id="findings-2" class="section level4">
<h4>Findings</h4>
<ul>
<li><p>COTA enhances GWAS interpretability by revealing trans-regulatory
networks.</p></li>
<li><p>New gene discoveries provide insights into disease
mechanisms.</p></li>
<li><p>Potential for targeted therapy development based on genetic
findings.</p></li>
</ul>
</div>
<div id="code" class="section level4">
<h4>Code</h4>
<ul>
<li><p>Codes for COTA are in <a
href="https://github.com/mxxptian/COTA/tree/d211df91c43826ae56adfa30b39d506716921ba7">COTA
Package.</a></p></li>
<li><p>Summary of codes are in <a
href="https://docs.google.com/document/d/1i0V_S7Yh-XlwvDcbUzki3WhIxmeUiaPs_Eazlr4lhys/edit?tab=t.0">code
summary.</a></p></li>
</ul>
</div>
</div>
</div>
<div id="data" class="section level2">
<h2>Data</h2>
<div id="whole-exon-sequencing" class="section level3">
<h3>Whole Exon Sequencing</h3>
</div>
<div id="trans-effect" class="section level3">
<h3>Trans Effect</h3>
</div>
<div id="gwas" class="section level3">
<h3>GWAS</h3>
<ol style="list-style-type: decimal">
<li><p>Numerized item 1</p>
<ol style="list-style-type: decimal">
<li><p>Numerized item 1.1</p></li>
<li><p>Numerized item 1.2</p></li>
</ol></li>
<li><p>Numerized item 2</p></li>
</ol>
</div>
<div id="insert-figures" class="section level3">
<h3>3. Insert figures</h3>
<div id="way-1" class="section level4">
<h4>Way 1</h4>
<p><img src="assets/logo.png" style="width:10.0%" /></p>
</div>
<div id="way-2" class="section level4">
<h4>Way 2</h4>
<pre class="r"><code>knitr::include_graphics(c(&quot;assets/logo.png&quot;), error = FALSE)</code></pre>
<div class="figure" style="text-align: center">
<img src="assets/logo.png" alt="Figure: Logo." width="10%" />
<p class="caption">
Figure: Logo.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-logo">
Past versions of logo.png
</button>
</p>
<div id="fig-logo" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/b1b4ba321e166675b9945cd602c9ba0e02636e93/docs/assets/logo.png" target="_blank">b1b4ba3</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="insert-tables" class="section level3">
<h3>4. Insert tables</h3>
<ul>
<li>By Markdown syntax</li>
</ul>
<table style="width:65%;">
<colgroup>
<col width="31%" />
<col width="33%" />
</colgroup>
<tbody>
<tr>
<td>AAA</td>
<td>123</td>
</tr>
<tr>
<td>bbb</td>
<td><ul>
<li>ccc: 123</li>
<li>ddd: 123</li>
<li>eee: 123</li>
<li>fff: 123</li>
<li>ggg: 123</li>
</ul></td>
</tr>
</tbody>
</table>
<ul>
<li>By R script</li>
</ul>
<pre class="r"><code>res = data.frame(&quot;aaa&quot; = c(123, 456),
                 &quot;bbb&quot; = c(123, 456),
                 &quot;ccc&quot; = c(123, 456),
                 stringsAsFactors = FALSE, check.names = FALSE)
knitr::kable(res)</code></pre>
<table>
<thead>
<tr>
<th align="right">aaa</th>
<th align="right">bbb</th>
<th align="right">ccc</th>
</tr>
</thead>
<tbody>
<tr>
<td align="right">123</td>
<td align="right">123</td>
<td align="right">123</td>
</tr>
<tr>
<td align="right">456</td>
<td align="right">456</td>
<td align="right">456</td>
</tr>
</tbody>
</table>
<ul>
<li>By reading a file</li>
</ul>
<pre class="r"><code>res = read.table(&#39;./data/file_res.txt&#39;, header = TRUE, sep = &quot;\t&quot;, stringsAsFactors = TRUE)
knitr::kable(res)</code></pre>
<table>
<thead>
<tr>
<th align="right">aaa</th>
<th align="right">bbb</th>
<th align="right">ccc</th>
</tr>
</thead>
<tbody>
<tr>
<td align="right">123</td>
<td align="right">123</td>
<td align="right">123</td>
</tr>
<tr>
<td align="right">456</td>
<td align="right">456</td>
<td align="right">456</td>
</tr>
</tbody>
</table>
</div>
<div id="insert-links" class="section level3">
<h3>5. Insert links</h3>
<ul>
<li>Link to a website</li>
</ul>
<p>Link to <a href="https://www.google.com/">google</a> by
<code>[google](https://www.google.com/)</code>.</p>
<ul>
<li>Link to a file</li>
</ul>
<p>Link to <a href="assets/logo.png">file</a> by
<code>[file](docs/assets/logo.png)</code>.</p>
<ul>
<li>Link to a header</li>
</ul>
<p>Link to <a href="2022.html#header-of-another-section">another
header</a> by
<code>[another header](2022.html#header-of-another-section)</code>.</p>
</div>
</div>
<div id="header-of-another-section" class="section level2">
<h2>Another week</h2>
<div id="analysis-1" class="section level3">
<h3>1. Analysis 1</h3>
<div id="analysis" class="section level4">
<h4>Analysis</h4>
</div>
<div id="figures-tables" class="section level4">
<h4>Figures, tables</h4>
</div>
<div id="observations" class="section level4">
<h4>Observations</h4>
</div>
</div>
<div id="analysis-2" class="section level3">
<h3>2. Analysis 2</h3>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.2 (2022-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Ventura 13.3.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.12      highr_0.10       compiler_4.2.2   pillar_1.10.1   
 [5] bslib_0.5.0      later_1.3.1      git2r_0.33.0     jquerylib_0.1.4 
 [9] tools_4.2.2      getPass_0.2-4    digest_0.6.31    jsonlite_1.8.8  
[13] evaluate_0.21    lifecycle_1.0.4  tibble_3.2.1     pkgconfig_2.0.3 
[17] rlang_1.1.3      cli_3.6.2        rstudioapi_0.14  yaml_2.3.7      
[21] xfun_0.39        fastmap_1.1.1    httr_1.4.7       stringr_1.5.1   
[25] knitr_1.43       fs_1.6.2         vctrs_0.6.5      sass_0.4.6      
[29] rprojroot_2.0.4  glue_1.7.0       R6_2.5.1         processx_3.8.1  
[33] rmarkdown_2.22   callr_3.7.3      magrittr_2.0.3   whisker_0.4.1   
[37] ps_1.7.5         promises_1.2.0.1 htmltools_0.5.5  httpuv_1.6.11   
[41] stringi_1.8.4    cachem_1.0.8    </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
