<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>2025.knit</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Research Progress</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Progress
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="2025.html">2025</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/zq2209/GradPLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-10-29
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradPLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220610code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220610code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220610)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtreed56a60ca054069a841db0e9b3dafeba1e68a5617targetblankd56a60ca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/zq2209/GradPLog/tree/d56a60ca054069a841db0e9b3dafeba1e68a5617" target="_blank">d56a60c</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtreed56a60ca054069a841db0e9b3dafeba1e68a5617targetblankd56a60ca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/zq2209/GradPLog/tree/d56a60ca054069a841db0e9b3dafeba1e68a5617" target="_blank">d56a60c</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.Rhistory

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/2025.Rmd</code>) and HTML
(<code>docs/2025.html</code>) files. If you’ve configured a remote Git
repository (see <code>?wflow_git_remote</code>), click on the hyperlinks
in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a76a6dc44a3803060125e7b705bc48ddf540743d/docs/2025.html" target="_blank">a76a6dc</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/33224c2f2bcc3284fc9ddc09e0e93550bff0bd69/analysis/2025.Rmd" target="_blank">33224c2</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-29
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a048dbb35a98a4b237bd8f02c4653b8935de9a11/docs/2025.html" target="_blank">a048dbb</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/1bbc88d93dc468414ffbc8b8ebafc2b091f7876a/analysis/2025.Rmd" target="_blank">1bbc88d</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/1bbc88d93dc468414ffbc8b8ebafc2b091f7876a/docs/2025.html" target="_blank">1bbc88d</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/c7099345fbc7343b7ec78b8ed9c4e04e74193b6d/analysis/2025.Rmd" target="_blank">c709934</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/c7099345fbc7343b7ec78b8ed9c4e04e74193b6d/docs/2025.html" target="_blank">c709934</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/2e5010c08cee1f2c4222b875c355b890ced7e6d1/analysis/2025.Rmd" target="_blank">2e5010c</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-28
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/b68a6ce4584ff9ef8016c643675a0c453106ab7b/docs/2025.html" target="_blank">b68a6ce</a>
</td>
<td>
“John
</td>
<td>
2025-10-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/933b6ae8c37e890aeffe2fb9f4646eb3926fb199/analysis/2025.Rmd" target="_blank">933b6ae</a>
</td>
<td>
“John
</td>
<td>
2025-10-22
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/97da75eb4d5153ff21448bf9f667135fd4db85c5/docs/2025.html" target="_blank">97da75e</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/851497d94a284a26df10ecb89876e470be95cfb5/analysis/2025.Rmd" target="_blank">851497d</a>
</td>
<td>
zq2209
</td>
<td>
2025-10-22
</td>
<td>
add jurakt perturb
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/d387fd36c2feebbb1579b4da58b721d9e2d3dacc/docs/2025.html" target="_blank">d387fd3</a>
</td>
<td>
zq2209
</td>
<td>
2025-09-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/7a73d1a9faf8ad7282123e3c330932726ae709b3/analysis/2025.Rmd" target="_blank">7a73d1a</a>
</td>
<td>
zq2209
</td>
<td>
2025-09-24
</td>
<td>
add k562 relavence analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/ad95a9e2b35f0b0db1bbef09161a730b9ea4510c/docs/2025.html" target="_blank">ad95a9e</a>
</td>
<td>
“John
</td>
<td>
2025-05-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/18e165c01428f5f0c81b7489a9ae2bbafffd96aa/analysis/2025.Rmd" target="_blank">18e165c</a>
</td>
<td>
“John
</td>
<td>
2025-05-20
</td>
<td>
update progression
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/9d8f4e002bceae74d66f9cbb0d18295b616bad2c/docs/2025.html" target="_blank">9d8f4e0</a>
</td>
<td>
“John
</td>
<td>
2025-05-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/6cf4626e425da725ea0147d01929815b4c40e217/analysis/2025.Rmd" target="_blank">6cf4626</a>
</td>
<td>
“John
</td>
<td>
2025-05-20
</td>
<td>
update progression
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a5cd70b230f51b2ca3664c359e6e17f485320a02/docs/2025.html" target="_blank">a5cd70b</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/a0a2f09389ab52a4ce58bea74122881ca79bbbce/analysis/2025.Rmd" target="_blank">a0a2f09</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a0a2f09389ab52a4ce58bea74122881ca79bbbce/docs/2025.html" target="_blank">a0a2f09</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/4454b4cabd37407bb93159e7cf3fed1664a22920/analysis/2025.Rmd" target="_blank">4454b4c</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/4454b4cabd37407bb93159e7cf3fed1664a22920/docs/2025.html" target="_blank">4454b4c</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/b0d85843205f8e19a3965d860274edc7f5020388/analysis/2025.Rmd" target="_blank">b0d8584</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/ed904209e7a5029befa089d77448e9c20167efd9/docs/2025.html" target="_blank">ed90420</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/d50c9285be2db62dbc038e3d8a4db43b41a7f68c/analysis/2025.Rmd" target="_blank">d50c928</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/d50c9285be2db62dbc038e3d8a4db43b41a7f68c/docs/2025.html" target="_blank">d50c928</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/f0563f35329866038c30502f35cc448750268608/analysis/2025.Rmd" target="_blank">f0563f3</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/f0563f35329866038c30502f35cc448750268608/docs/2025.html" target="_blank">f0563f3</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/0b4a87deeca7d2cb1adfca846cd8894ee5fa5e86/analysis/2025.Rmd" target="_blank">0b4a87d</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/431eccccda41602de7a041cf3c75311ee5493891/docs/2025.html" target="_blank">431eccc</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/9f34755bf39c78d790531444ddcd248a55a11f43/analysis/2025.Rmd" target="_blank">9f34755</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/9f34755bf39c78d790531444ddcd248a55a11f43/docs/2025.html" target="_blank">9f34755</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/b26e62d562d6de757116041774b136a015118f8b/analysis/2025.Rmd" target="_blank">b26e62d</a>
</td>
<td>
“John
</td>
<td>
2025-02-19
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/885485521c634a9600121ac3ea25d838e85b8fa4/docs/2025.html" target="_blank">8854855</a>
</td>
<td>
“John
</td>
<td>
2025-02-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/103cb30a5da4b24ee376579bca2422a299a367c3/analysis/2025.Rmd" target="_blank">103cb30</a>
</td>
<td>
“John
</td>
<td>
2025-02-11
</td>
<td>
build site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/296c9e26a338f8007bc4ac595a8e4de42f15d35a/docs/2025.html" target="_blank">296c9e2</a>
</td>
<td>
“John
</td>
<td>
2025-02-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/15f4806f4d42127b970fedefe07983434cd5d8cc/analysis/2025.Rmd" target="_blank">15f4806</a>
</td>
<td>
“John
</td>
<td>
2025-02-10
</td>
<td>
build site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/70deb4c897017f193c628aacb093824cdd79adcd/docs/2025.html" target="_blank">70deb4c</a>
</td>
<td>
“John
</td>
<td>
2025-02-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/1dc0e4de18987d7720f14050abab73dab4f61444/docs/2025.html" target="_blank">1dc0e4d</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/315b26cb12a7b7a88973f6daa5e196861bd645a5/analysis/2025.Rmd" target="_blank">315b26c</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/86817dcf466da2e2fa444ff98f72265ffa4fd775/docs/2025.html" target="_blank">86817dc</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/4400a1eb44fea65964de9903e8b9a561761cd17d/docs/2025.html" target="_blank">4400a1e</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/a04e379e88c4d972ad67d9d2245f268bbe76d2ae/analysis/2025.Rmd" target="_blank">a04e379</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a04e379e88c4d972ad67d9d2245f268bbe76d2ae/docs/2025.html" target="_blank">a04e379</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/7b92cf46be1f0fa7b5f89cd7ce61237944417929/analysis/2025.Rmd" target="_blank">7b92cf4</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/7b92cf46be1f0fa7b5f89cd7ce61237944417929/docs/2025.html" target="_blank">7b92cf4</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/44013b94a2a057ef650b4d3c65f42cfee813515b/analysis/2025.Rmd" target="_blank">44013b9</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/44013b94a2a057ef650b4d3c65f42cfee813515b/docs/2025.html" target="_blank">44013b9</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/c405b4f924b7514cd098a36be6d3fcb377ced7b3/analysis/2025.Rmd" target="_blank">c405b4f</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/c405b4f924b7514cd098a36be6d3fcb377ced7b3/docs/2025.html" target="_blank">c405b4f</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/910013fff1a61f3be0bdd47a6d42f950b7748e61/analysis/2025.Rmd" target="_blank">910013f</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/46881214a06952249381cf398370706cebe008c8/analysis/2025.Rmd" target="_blank">4688121</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/46881214a06952249381cf398370706cebe008c8/docs/2025.html" target="_blank">4688121</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/927813ba777c8584121380da9fe875ca0c8413d3/analysis/2025.Rmd" target="_blank">927813b</a>
</td>
<td>
“John
</td>
<td>
2025-02-04
</td>
<td>
build_site
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/dd58251b25b5fb042bd64723f41fe9ae216cc20a/docs/2025.html" target="_blank">dd58251</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/f5e82f5bd2d2ecebca2e3ce7581dac18f6b96803/docs/2025.html" target="_blank">f5e82f5</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/ef8dd5ed5702dce4ddd342b9aee74db4bf4a08f0/docs/2025.html" target="_blank">ef8dd5e</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/63b8b33ae7db95ea9c4e86ac5711d237d52620b1/analysis/2025.Rmd" target="_blank">63b8b33</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/ef2284b7dcecf044b15a4a11cee3f6505d182518/docs/2025.html" target="_blank">ef2284b</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/bcff4bda3b9dfbf083b60415ab74bea271dcb5e3/docs/2025.html" target="_blank">bcff4bd</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/ded2ce1b7f2e007c613904c6ff75933caac3c086/analysis/2025.Rmd" target="_blank">ded2ce1</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
build sites
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/0a7be27441d1d00773bd95766d1f3cfad037eb41/docs/2025.html" target="_blank">0a7be27</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/76ffb2c2774fdf4e893c137ba0402faac7871eb7/analysis/2025.Rmd" target="_blank">76ffb2c</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/76ffb2c2774fdf4e893c137ba0402faac7871eb7/docs/2025.html" target="_blank">76ffb2c</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/2d02830e75153cd4b77c0caea5e58521914ce63d/analysis/2025.Rmd" target="_blank">2d02830</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/2d02830e75153cd4b77c0caea5e58521914ce63d/docs/2025.html" target="_blank">2d02830</a>
</td>
<td>
“John
</td>
<td>
2025-02-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/effd6aa650b2cea68f2b60211c07d6e9814cdb7a/analysis/2025.Rmd" target="_blank">effd6aa</a>
</td>
<td>
“John
</td>
<td>
2025-02-01
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/390b75e3d99f0c1a4e6b25cd664445772f998efa/analysis/2025.Rmd" target="_blank">390b75e</a>
</td>
<td>
“John
</td>
<td>
2025-01-31
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/390b75e3d99f0c1a4e6b25cd664445772f998efa/docs/2025.html" target="_blank">390b75e</a>
</td>
<td>
“John
</td>
<td>
2025-01-31
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="week-1-jan-6---jan-12" class="section level2">
<h2>Week 1: Jan 6 - Jan 12</h2>
<div id="current-research-on-ibd" class="section level3">
<h3>Current Research on IBD</h3>
<p>This section summarize the WES and GWAS studies on IBD, giving
background information on the disease and current research on the
diease.</p>
<div id="disease-background" class="section level4">
<h4>Disease Background</h4>
<div id="general-background-of-ibd" class="section level5">
<h5>General Background of IBD</h5>
<p><img src="assets/image1.png" style="width:60.0%" /></p>
<ul>
<li><p>IBD is Chronic inflammation influenced by genetics, environment,
microbiota, and immunity.</p></li>
<li><p>Genetic Contribution</p>
<ul>
<li><p>Crohn’s Disease (CD): 15% family history; twin studies show 50%
concordance in monozygotic (MZ) twins vs. less than 10% in dizygotic
(DZ) twins.</p></li>
<li><p>GWAS identified 163 loci; trans-ethnic studies identified an
additional 38 loci.</p></li>
</ul></li>
<li><p>Epigenetics</p>
<ul>
<li><p>Genome-environment interactions affect disease
progression.</p></li>
<li><p>Emerging research focuses on the role of epigenetics in
IBD.</p></li>
</ul></li>
</ul>
</div>
<div id="genetic-associations-in-ibd" class="section level5">
<h5>Genetic Associations in IBD</h5>
<ul>
<li><p>NOD2: First CD-associated gene (2001), with key variants R702W
and G908R.</p></li>
<li><p>Autophagy Genes: ATG16L1, LRRK2, and IRGM predispose individuals
to IBD.</p></li>
<li><p>IL-10 Receptor Mutations: IL10RA, IL10RB are linked to
colitis.</p></li>
<li><p>IBD-Associated Loci: ~240 loci identified (as of 2022); 30 shared
between CD and UC.</p></li>
<li><p>CD Predictive Loci: FOXO3, IGFBP1, and XACT as potential
markers.</p></li>
</ul>
</div>
</div>
<div
id="large-scale-exome-sequencing-identifies-novel-genes-and-pathways-in-crohns-disease"
class="section level4">
<h4>Large-Scale Exome Sequencing Identifies Novel Genes and Pathways in
Crohn’s Disease</h4>
<p>Sazonovs, Aleksejs et al.</p>
<div id="motivation" class="section level5">
<h5>Motivation</h5>
<ul>
<li><p>CD is a chronic inflammatory disorder with a strong genetic
component.</p></li>
<li><p>GWAS has primarily focused on common variants, but rare coding
variants remain under-explored.</p></li>
</ul>
</div>
<div id="study-design" class="section level5">
<h5>Study Design</h5>
<p><img src="assets/image2.png" style="width:60.0%" /></p>
</div>
<div id="findings" class="section level5">
<h5>Findings</h5>
<ul>
<li>Single Variant Analysis: 94 out of 116 variants replicated in
discovery datasets.</li>
</ul>
<p><img src="assets/image3.png" style="width:80.0%" /> <img
src="assets/image4.png" style="width:60.0%" /></p>
<ul>
<li>Burden Test: Missense vs. frameshift variants analyzed.</li>
</ul>
<p><img src="assets/image5.png" style="width:90.0%" /></p>
</div>
<div id="takeaways" class="section level5">
<h5>Takeaways</h5>
<ul>
<li>Exome Sequencing Complements GWAS
<ul>
<li>Addresses gaps in genetic architecture (low-frequency and rare
variants) that earlier CD GWAS meta-analyses could not observe.</li>
</ul></li>
<li>Significance of Coding Variants:
<ul>
<li>Coding variants, although fewer than noncoding variants, are highly
enriched for associations with both common and rare diseases.</li>
<li>Tend to have stronger effects due to natural selection, which often
keeps their frequencies low.</li>
<li>Provide direct links to specific genes and pathogenic mechanisms,
unlike noncoding variants.</li>
</ul></li>
<li>Key Findings in CD Pathogenesis:
<ul>
<li>Novel genes such as PDLIM5, SDF2L1, HGFAC, PAF-R, and CCR7 emphasize
the role of mesenchymal cells (MCs) in intestinal inflammation.</li>
</ul></li>
<li>Therapeutic Implications:
<ul>
<li>Findings highlight the potential for therapeutic strategies
targeting mesenchymal niche balance to address CD pathogenesis.</li>
<li>Genetic evidence for drug targets is valuable for driving drug
development.</li>
</ul></li>
<li>Future Directions:
<ul>
<li>Expanded sequencing efforts in ulcerative colitis and integrated
analyses with larger GWAS studies are expected to identify more linked
genes.</li>
</ul></li>
</ul>
</div>
</div>
<div
id="genome-wide-association-study-implicates-immune-activation-of-multiple-integrin-genes-in-inflammatory-bowel-disease"
class="section level4">
<h4>Genome-wide association study implicates immune activation of
multiple integrin genes in inflammatory bowel disease</h4>
<p>de Lange, K., Moutsianas, L., Lee, J. et al.</p>
<div id="motivation-1" class="section level5">
<h5>Motivation</h5>
<ul>
<li><p>Current treatments involve immunomodulators, but patients often
experience side effects or treatment resistance.</p></li>
<li><p>GWAS and Immunochip studies have identified risk loci but have
had limited therapeutic impact.</p></li>
</ul>
</div>
<div id="study-design-1" class="section level5">
<h5>Study Design</h5>
<p><img src="assets/image6.png" style="width:90.0%" /></p>
</div>
<div id="findings-1" class="section level5">
<h5>Findings</h5>
<p>Identified 25 new GWAS loci</p>
<ul>
<li>4 loci with significant variants
<ul>
<li>SLAMF8</li>
<li>RORC</li>
<li>PLCG2</li>
<li>NCF4</li>
</ul></li>
<li>Another 4 loci within integrin gene
<ul>
<li>3 loci showed larger than 90% probability have colocalization with
eQTL</li>
<li>1 loci showed intermediate evidendence</li>
</ul></li>
</ul>
</div>
<div id="takeaways-1" class="section level5">
<h5>Takeaways</h5>
<ul>
<li><p>Integrins are not only important in cell trafficking but can also
participate in cellular signaling.</p></li>
<li><p>Highlighted integrins as key therapeutic targets:</p>
<ul>
<li>Monoclonal antibodies like vedolizumab and etrolizumab targeting
integrins have shown efficacy in IBD.</li>
<li>Identified SMAD7, a modulator of <span
class="math inline">\(TGF-\beta\)</span> signaling, as a potential
target for Crohn’s disease treatment.</li>
</ul></li>
<li><p>Emphasized the importance of gut-selective therapies to minimize
risks like progressive multifocal leukoencephalopathy (PML).</p></li>
<li><p>These discoveries have demonstrated that the effect sizes of GWAS
associations do not necessarily reflect the importance or therapeutic
relevance of their underlying biological pathways.</p></li>
</ul>
</div>
</div>
</div>
<div id="method" class="section level3">
<h3>Method</h3>
<div
id="prioritizing-disease-mediating-genes-leveraging-trans-gene-regulation"
class="section level4">
<h4>Prioritizing disease-mediating genes leveraging trans-gene
regulation</h4>
<div id="motivation-2" class="section level5">
<h5>Motivation</h5>
<ul>
<li>Complex Genetic Architecture of Diseases:
<ul>
<li>Complex traits are highly polygenic, involving numerous genes with
varying effect sizes.</li>
<li>The omnigenic model posits that all expressed genes in
disease-relevant tissues contribute to disease risk, but not all genes
have equal importance.</li>
</ul></li>
<li>Importance of Disease-Mediating Genes:
<ul>
<li>Disease-associated genes can be categorized into:
<ul>
<li>Disease-mediating genes: Directly affect the disease.</li>
<li>Peripheral genes: Indirectly influence the disease through
trans-regulatory networks.</li>
</ul></li>
<li>Identifying disease-mediating genes is critical for uncovering
pathways and mechanisms relevant to treatment development.</li>
</ul></li>
<li>Limitations of Current Gene Prioritization Methods:
<ul>
<li>GWAS:
<ul>
<li>Identifies important disease-mediating genes but may miss many due
to lack of power in detecting low-frequency or rare variants with large
effects.</li>
</ul></li>
<li>Rare Variant Methods:
<ul>
<li>Gene-based approaches, like burden tests in WES, improve detection
but are limited by current sample sizes.</li>
</ul></li>
<li>Graph and Canonical Correlation Models:
<ul>
<li>Recent approaches predict candidate genes or link expressions to
variants but fail to explicitly differentiate disease-mediating from
peripheral genes or reveal trans-regulatory networks.</li>
</ul></li>
</ul></li>
<li>Challenges with Cis-eQTLs in Disease Gene Discovery:
<ul>
<li>Disease genes often lack strong cis-QTL signals due to stronger
selective constraints.</li>
<li>Cis-eQTL limitations necessitate focusing on trans-QTLs, as their
target genes share similar selective constraints with disease
genes.</li>
</ul></li>
<li>Significance of Trans-Regulation:
<ul>
<li>Trans-regulatory networks explain a large portion (~70%) of disease
heritability.</li>
<li>Understanding trans-regulation provides insights into collaborative
gene contributions to disease risk and highlights the importance of
completing trans-gene regulatory networks for understanding complex
trait genetics.</li>
</ul></li>
</ul>
</div>
<div id="cota" class="section level5">
<h5>COTA</h5>
<ul>
<li><p>COTA Model: Integrates trans-regulatory effects to identify
disease-mediating genes.</p></li>
<li><p>GBAT: Predicts gene expression using machine learning
models.</p></li>
</ul>
<p><img src="assets/image7.png" style="width:80.0%" /></p>
<ul>
<li>DACT: Estimates gene effects from burden tests.</li>
</ul>
<p><img src="assets/image8.png" style="width:80.0%" /></p>
</div>
<div id="findings-2" class="section level5">
<h5>Findings</h5>
<ul>
<li><p>COTA enhances GWAS interpretability by revealing trans-regulatory
networks.</p></li>
<li><p>New gene discoveries provide insights into disease
mechanisms.</p></li>
<li><p>Potential for targeted therapy development based on genetic
findings.</p></li>
</ul>
</div>
<div id="code" class="section level5">
<h5>Code</h5>
<ul>
<li><p>Codes for COTA are in <a
href="https://github.com/mxxptian/COTA/tree/d211df91c43826ae56adfa30b39d506716921ba7">COTA
Package.</a></p></li>
<li><p>Summary of codes are in <a
href="https://docs.google.com/document/d/1i0V_S7Yh-XlwvDcbUzki3WhIxmeUiaPs_Eazlr4lhys/edit?tab=t.0">code
summary.</a></p></li>
</ul>
</div>
</div>
</div>
</div>
<div id="week-2-jan-13---jan-19" class="section level2">
<h2>Week 2: Jan 13 - Jan 19</h2>
<div id="data" class="section level3">
<h3>Data</h3>
<div id="whole-exon-sequencing" class="section level4">
<h4>Whole Exon Sequencing</h4>
<div id="files" class="section level5">
<h5>Files</h5>
<ul>
<li>UC: Burden tests with UC only</li>
<li>CD: Burden tests with CD only</li>
<li>IBD: Burden tests with other types of IBD</li>
</ul>
</div>
</div>
<div id="variants-analyzed" class="section level4">
<h4>Variants analyzed</h4>
<ul>
<li>NSYN: Non-synonymous Variants
<ul>
<li>nsyn.singleton: variants appear only once across all samples</li>
<li>nsyn.0.001: allele frequancy less than 0.001</li>
</ul></li>
<li>NSYN_AM: Non-synonymous variants with an AlphaMissense score greater
than 0.2
<ul>
<li>nsyn.singleton: variants appear only once across all samples</li>
<li>nsyn.0.001: allele frequancy less than 0.001</li>
</ul></li>
<li>pLoF: PTV variants that are also flagged as high confidence (HC) by
Loftee
<ul>
<li>pLoF.singleton: variants appear only once across all samples</li>
<li>pLoF.nsyn.0.001: allele frequancy less than 0.001</li>
</ul></li>
</ul>
<div id="variables" class="section level5">
<h5>Variables</h5>
<ul>
<li>MarkerName: Gene name</li>
<li>Allele1: Study allele</li>
<li>Allele2: Ref allele</li>
<li>Freq1: Allele frequency of Allele1</li>
<li>Effect: Effect size</li>
<li>StdErr: Standrad error of effect size</li>
<li>log(P): Log p-value of effect size</li>
<li>Direction: Direction of effect size from two platform</li>
<li>HetISq: Heterogeneity statistics I-square</li>
<li>HetChiSq: Chi-square statistics for heterogeneity that effect for
two platform are come from same distribution</li>
<li>HetDf: Degree of freedom</li>
<li>logHetP: Log p-value of HetChiSq</li>
</ul>
</div>
<div id="eda-of-data" class="section level5">
<h5>EDA of data</h5>
<ul>
<li>Basic visualizations of <strong>p-values</strong> were performed,
including <strong>QQ plots</strong> and <strong>histograms</strong>, for
burden tests involving <strong>singleton</strong> and <strong>&lt;0.001
NSYN/LoF variants</strong> in <strong>Crohn’s Disease (CD)</strong>.
<ul>
<li>The overall distribution of <strong>p-values</strong> for <strong>CD
with NSYN and LoF variants</strong> appears nearly identical.<br />
</li>
<li>Using a uniform distribution, the points predominantly align with
the line, exhibiting slight curvature.</li>
</ul></li>
</ul>
<p><img src="assets/image10.png" style="width:50.0%" /></p>
<p><img src="assets/image11.png" style="width:50.0%" /></p>
<ul>
<li>Significant heterogeneity across all <strong>nine studies</strong>
was assessed.
<ul>
<li>Two genes met the significance threshold of
<strong>0.05/20,000</strong> (<strong>total genes: 18,641</strong>):
<ul>
<li><strong>SULT1A1 (Sulfotransferase Family 1A Member 1):</strong>
Encodes sulfotransferase enzymes that facilitate <strong>sulfate
conjugation</strong> of various hormones, neurotransmitters, drugs, and
xenobiotic compounds. These cytosolic enzymes differ in <strong>tissue
distribution</strong> and <strong>substrate specificity</strong>.<br />
</li>
<li><strong>PLXND1 (Plexin D1):</strong> Enables <strong>protein
domain-specific binding activity</strong> and is involved in
<strong>axonogenesis regulation</strong>, <strong>semaphorin-plexin
signaling</strong>, and <strong>synapse assembly</strong>. Also plays a
role in <strong>circulatory system development</strong>,
<strong>salivary gland branching</strong>, and <strong>kidney
development</strong>, and is primarily located in
<strong>lamellipodia</strong>.<br />
</li>
</ul></li>
<li>The paper reported <strong>43 sites</strong> with significant
heterogeneity, whereas only <strong>two genes</strong> were identified
in this analysis.<br />
</li>
<li>The original study applied a <strong>heterogeneity-of-effect
test</strong> between the <strong>Nextera</strong> and <strong>Twist
discovery cohorts</strong> using <strong>IVW PHET &lt; 0.0001</strong>
as the threshold.<br />
</li>
<li>The discrepancy may be due to <strong>differences in heterogeneity
statistics</strong> and a <strong>more lenient p-value cutoff</strong>
in this analysis.<br />
</li>
<li>When <strong>0.0001</strong> was used as the threshold, <strong>32
genes</strong> exhibited significant heterogeneity, including
<strong>SULT1A1 and PLXND1</strong>.</li>
</ul></li>
<li><strong>Basic summary of genes in the WES dataset</strong>:
<ul>
<li><strong>18,641 genes</strong> are present in the dataset.<br />
</li>
<li><strong>15,309 genes</strong> are included in <strong>all nine
studies</strong>.<br />
</li>
<li><strong>3,332 genes</strong> are shared across some but not all
studies.<br />
</li>
<li>In the <strong>CD with NSYN 0.01 study</strong>, <strong>64
genes</strong> lack <strong>burden tests on singleton
variants</strong>.<br />
</li>
<li>Some genes do not have variants that appear in only one sample.</li>
</ul></li>
</ul>
<p><img src="assets/image12.png" style="width:80.0%" /></p>
<p><img src="assets/image13.png" style="width:80.0%" /></p>
</div>
</div>
<div id="trans-effect" class="section level4">
<h4>Trans Effect</h4>
<ul>
<li>trans-eQTL:
<ul>
<li>eQTLGen</li>
<li>Whole blood</li>
</ul></li>
<li>GBAT</li>
</ul>
</div>
<div id="gwas" class="section level4">
<h4>GWAS</h4>
<ul>
<li><p>The dataset consists of <strong>29,849 East Asian
samples</strong> from China, Hong Kong SAR, Japan, and the Republic of
Korea, and <strong>368,819 European samples</strong> from the U.S., NR,
and Finland.</p></li>
<li><p>The total number of <strong>cases</strong> in the dataset is
<strong>45,106</strong>, while the total number of
<strong>controls</strong> is <strong>353,562</strong>.</p></li>
<li><p>Among the <strong>East Asian cohort</strong>, there are
<strong>14,393 cases</strong> and <strong>15,456
controls</strong>.</p></li>
<li><p>Among the <strong>European cohort</strong>, there are
<strong>30,713 cases</strong> and <strong>338,106
controls</strong>.</p></li>
<li><p>The majority of cases are diagnosed with <strong>Crohn’s Disease
(CD)</strong> and <strong>Ulcerative Colitis (UC)</strong>, with a
smaller number of <strong>other Inflammatory Bowel Disease (IBD)
cases</strong>.</p></li>
<li><p>The <strong>East Asian samples represent approximately
7.5%</strong> of the total dataset.</p></li>
</ul>
<p><img src="assets/image9.png" style="width:80.0%" /></p>
</div>
</div>
</div>
<div id="week-3-jan-20---jan-26" class="section level2">
<h2>Week 3: Jan 20 - Jan 26</h2>
<div id="analysis" class="section level3">
<h3>Analysis</h3>
<div id="data-1" class="section level4">
<h4>Data</h4>
<div id="wes-data" class="section level5">
<h5>WES Data</h5>
<ul>
<li>CD with Variant Frequency less than 0.001</li>
<li>Total Genes: 18,620</li>
</ul>
</div>
<div id="genes-detected-by-wes" class="section level5">
<h5>Genes Detected by WES</h5>
<ul>
<li>Total of 32 significant genes.</li>
<li><strong>Key Genes Identified</strong>:
<ul>
<li><strong>NOD2</strong>: Plays a role in <strong>autophagy</strong>,
aiding in the destruction of bacteria, viruses, and harmful
substances.</li>
<li><strong>SBNO2</strong>: Differentially regulated by
<strong>lipopolysaccharide and IL-10</strong> (<a
href="https://www.nature.com/articles/s41467-024-47218-3">Nature
Link</a>).</li>
<li><strong>ATG4C</strong>: Involved in <strong>autophagy</strong>, the
process of degrading proteins and organelles.</li>
<li><strong>IGHV Family</strong>: Immunoglobulin heavy variable
genes.</li>
<li><strong>AHNAK2</strong>: Encodes a <strong>large
nucleoprotein</strong>.</li>
</ul></li>
</ul>
</div>
</div>
<div id="snp-based-analysis" class="section level4">
<h4>SNP-Based Analysis</h4>
<ul>
<li>GWAS Significant SNPs (<em>p</em> &lt; 1e-4): 69,654 SNPs
<ul>
<li>Overlap with trans-eQTL SNPs: 1033 SNPs</li>
</ul></li>
<li>Total Trans-eQTL SNPs: 10,317</li>
</ul>
</div>
<div id="gene-based-analysis" class="section level4">
<h4>Gene-Based Analysis</h4>
<ul>
<li>GWAS Significant Genes** (<em>p</em> &lt; 1e-4): 69,654 SNPs
<ul>
<li>Overlap of GWAS Nearest Gene with GBAT trans regulation target:
2134</li>
</ul></li>
<li>Total Trans-eQTL Gene1: 13,634</li>
</ul>
</div>
</div>
</div>
<div id="week-4-jan-27---feb-2" class="section level2">
<h2>Week 4: Jan 27 - Feb 2</h2>
<div id="findings-3" class="section level3">
<h3>Findings</h3>
<div id="genes-detected-by-cota-snp-based" class="section level4">
<h4>Genes Detected by COTA (SNP-Based)</h4>
<ul>
<li><p><strong>Significant Gene Pairs</strong>:</p>
<ul>
<li>Shown in <a href="">significant gene pairs</a></li>
</ul></li>
<li><p><strong>WES Significant Gene2</strong>: NOD2, SBNO2</p></li>
<li><p><strong>WES Non-Significant Gene2</strong>: CARD9, CEACAM8,
USP36</p></li>
<li><p><strong>Network of detected gene</strong>:</p></li>
</ul>
<p><img src="assets/network_snp.png" style="width:100.0%" /></p>
</div>
<div id="genes-detected-by-cota-gene-based" class="section level4">
<h4>Genes Detected by COTA (Gene-Based)</h4>
<ul>
<li><p><strong>Significant Gene Pairs</strong>:</p>
<ul>
<li>Shown in <a href="">significant gene pairs</a></li>
</ul></li>
<li><p><strong>WES Significant Gene2</strong>: NOD2, AHNAK2</p></li>
<li><p><strong>WES Non-Significant Gene2</strong>: SH3YL1, TMED6,
CEP104, SPIRE2, ING1, EGLN3, PLEKHO2, EML4, ALOX5</p></li>
<li><p><strong>Network of detected gene</strong>:</p></li>
</ul>
<p><img src="assets/network_gene.png" style="width:100.0%" /></p>
</div>
</div>
</div>
<div id="week-5-feb-3---feb-9" class="section level2">
<h2>Week 5: Feb 3 - Feb 9</h2>
<div id="perturb-seq-data" class="section level3">
<h3>Perturb-seq data</h3>
<p>How does the data generated</p>
<ul>
<li><p>what is cell line</p></li>
<li><p>trans regulation target, gene_id?</p>
<ul>
<li>direction of correlation, which one regulate which?</li>
</ul></li>
<li><p>overlap of burden test significant genes and target genes is
larger</p>
<ul>
<li><p>13407 total regulator, and all 32 burden test genes are in
regulator</p></li>
<li><p>4165 of trans target, 20 burden test genes are in trans
target</p></li>
</ul></li>
<li><p>QTL calculation</p>
<ul>
<li><p>why negative binomial</p>
<ul>
<li>Count data that over dispersed?</li>
</ul></li>
</ul></li>
</ul>
</div>
</div>
<div id="week-6-feb-10---feb-16---discussion" class="section level2">
<h2>Week 6: Feb 10 - Feb 16 - Discussion</h2>
<div
id="why-most-of-burden-tests-significant-genes-are-not-detected-by-cota"
class="section level3">
<h3>Why most of burden tests significant genes are not detected by
COTA</h3>
<div
id="check-the-overlap-of-burden-tests-significant-genes-and-trans-regulation-target"
class="section level4">
<h4>Check the overlap of burden tests significant genes and trans
regulation target</h4>
<ul>
<li>SNP based
<ul>
<li>5 significant genes from the burden tests (AHNAK2, DNMT3A, NOD2,
SBNO2, ATG4C) overlap with the trans-eQTL genes used for analysis.</li>
</ul></li>
</ul>
<p><img src="assets/image14.png" style="width:60.0%" /></p>
<ul>
<li>Gene based
<ul>
<li>6 significant genes from the burden tests (AHNAK2, DNMT3A, NOD2,
SBNO2, ATG4C) overlap with the trans regulation targets used for
analysis.</li>
</ul></li>
</ul>
<p><img src="assets/image15.png" style="width:60.0%" /></p>
</div>
<div id="check-dgn-matrix" class="section level4">
<h4>Check DGN matrix</h4>
<p>There are still 5 significant burden test genes (AHNAK2, DNMT3A,
NOD2, SBNO2, ATG4C) overlapped with DGN.</p>
</div>
<div id="check-gtex" class="section level4">
<h4>Check GTEx</h4>
<ul>
<li><p>Bulk gene expression have all the significant burden test
genes.</p></li>
<li><p>Missed burden test significant genes</p>
<ul>
<li><p>immunoglobulin heavy joining</p>
<ul>
<li>IGHJ4 (immunoglobulin heavy joining 4; chr14:105864215-105864260:-):
Highly expressed at Cells - EBV-transformed lymphocytes and Spleen,
intermediate expression in Whole blood</li>
</ul>
<p><img src="assets/IGHJ4_tpm.png" style="width:80.0%" /></p></li>
<li><p>immunoglobulin heavy variable</p>
<ul>
<li>IGHV4-34 (immunoglobulin heavy variable 4-34;
chr14:106373663-106374145:-): Highly expressed at Cells -
EBV-transformed lymphocytes</li>
</ul>
<p><img src="assets/IGHV4-34_tpm.png" style="width:80.0%" /></p></li>
<li><p>immunoglobulin kappa joining</p>
<ul>
<li>IGKJ1 (immunoglobulin kappa joining 1; chr2:88861886-88861923:-):
Highly expressed at Cells - EBV-transformed lymphocytes, intermediate
expression in Spleen and Colon - Transverse, Small Intestine - Terminal
Ileum, Minor Salivary Gland</li>
</ul>
<p><img src="assets/IGKJ1_tpm.png" style="width:80.0%" /></p></li>
<li><p>immunoglobulin kappa variable</p></li>
<li><p>IGKV4-1 (immunoglobulin kappa variable 4-1;
chr2:88885397-88886153:+): Highly expressed at Cells - EBV-transformed
lymphocytes</p></li>
</ul>
<p><img src="assets/IGKV4-1_tpm.png" style="width:80.0%" /></p>
<ul>
<li>immunoglobulin heavy diversity
<ul>
<li>IGHD3-16 (immunoglobulin heavy diversity 3-16;
chr14:105895634-105895670:-): Highly expressed at Lung, Cells -
EBV-transformed lymphocytes, and Spleen</li>
</ul>
<img src="assets/IGHD3-16_tpm.png" style="width:80.0%" /></li>
</ul></li>
<li><p>TNFRSF6B (TNF receptor superfamily member 6b;
chr20:63696652-63698641:+): Highly expressed in Lung???</p></li>
</ul>
<p><img src="assets/TNFRSF6B_tpm.png" style="width:80.0%" /></p>
<ul>
<li><p>Overlapped burden test significant genes</p>
<ul>
<li>NOD2 (nucleotide binding oligomerization domain containing 2;
chr16:50693588-50734041:+): Expression in Whole Blood are higher than
others</li>
</ul>
<p><img src="assets/NOD2_tpm.png" style="width:80.0%" /></p>
<ul>
<li>ATG4C (autophagy related 4C cysteine peptidase;
chr1:62784132-62865516:+): Expressed in all tissues, higher expression
in brain tissue</li>
</ul>
<p><img src="assets/ATG4C_tpm.png" style="width:80.0%" /></p>
<ul>
<li>SBNO2 (strawberry notch homolog 2; chr19:1107637-1174268:-):
Expression in Whole Blood are higher than others</li>
</ul>
<p><img src="assets/SBNO2_tpm.png" style="width:80.0%" /></p>
<ul>
<li>AHNAK2 (AHNAK nucleoprotein 2; chr14:104937762-104978374:-): Higher
expression in skin tissue then whole blood</li>
</ul>
<p><img src="assets/AHNAK2_tpm.png" style="width:80.0%" /></p>
<ul>
<li>DNMT3A (DNA methyltransferase 3 alpha; chr2:25227855-25342590:-):
Expressed in all tissues, no really extream TPM values in some
tissue</li>
</ul>
<p><img src="assets/DNMT3A_tpm.png" style="width:80.0%" /></p></li>
<li><p>Check TPM of genes on bulk data</p>
<ul>
<li>Near all gene expression for those missed genes in trans effect are
higher, probably due to have extreme high TPM in some tissues, whereas
those overlapped genes don’t have such extreme expression.</li>
</ul>
<p><img src="assets/bulk_tpm.png" style="width:80.0%" /></p></li>
</ul>
</div>
</div>
<div
id="why-there-are-many-more-significant-genes-than-sazonovs-et-al-2022"
class="section level3">
<h3>Why there are many more significant genes than Sazonovs et al
2022</h3>
<ul>
<li><p>The data shared to us are different than 2022, we have newer
version of their results.</p>
<ul>
<li><p>Samples are different than what used in 2022 paper, the sample
size are similar but samples are different.</p></li>
<li><p>New exome capture method that could capture exome region
previously not included.</p></li>
</ul></li>
<li><p>Genes may also be not totally consistent with 2022</p>
<ul>
<li>Version of gene annotation are different in two platform</li>
</ul></li>
<li><p>Immunoglobulin genes are filtered out in the analysis by thinking
it as artifact</p>
<ul>
<li><p>The samples in the consortium come from different
hospitals</p></li>
<li><p>There is a possibility that a small number of immortalized cell
line samples may have been mixed in.<br />
</p></li>
<li><p>These cell lines are derived from some B cells, and
immunoglobulin gene region in B cells is highly recombinant.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="week-7-feb-17---feb-23" class="section level2">
<h2>Week 7: Feb 17 - Feb 23</h2>
<div id="integrate-perturb-data-with-cota" class="section level3">
<h3>Integrate perturb data with COTA</h3>
<div id="preprocessing-data" class="section level4">
<h4>Preprocessing data</h4>
<ul>
<li><p>Originally 14680 targets; 4734 regulators</p></li>
<li><p>Apply two steps filter on genes</p>
<ul>
<li><p>Filter out genes that don’t have a matched gene name</p>
<ul>
<li>drop 1536 targets and 1 regulator</li>
</ul></li>
<li><p>Select regulators that are GWAS gens (nearest genes of GWAS
loci)</p>
<ul>
<li>12730 targets and 1337 regulators remain for analysis</li>
</ul></li>
</ul></li>
<li><p>Among 12370 targets, 4 WES significant genes were included.</p>
<ul>
<li><p>DNMT3A, TNFRSF6B, ATG4C, SBNO2</p></li>
<li><p>NOD2 and AHNAK2 were not included in targets</p></li>
</ul></li>
<li><p>NAs in the p-value matrix of 12730 targets (rows) and 1337
regulators (columns)</p>
<ul>
<li><p>These pairs (regulator - target) don’t exist in the perturb-seq
result</p></li>
<li><p>Fill in with 0.99 to make sure COTA analysis can be
applied</p></li>
</ul></li>
</ul>
</div>
<div id="visualization-of-pvalue" class="section level4">
<h4>Visualization of pvalue</h4>
<ul>
<li><p>Basic visualizations of <strong>p-values</strong> were performed,
including <strong>QQ plots</strong> and <strong>histograms</strong>, for
all the gene pairs and filtered gene pairs (GWAS pairs).</p>
<ul>
<li><p>The overall distribution of <strong>p-values</strong> for all
pairs and GWAS pairs appears nearly identical.</p></li>
<li><p>Using a uniform distribution, the points predominantly align with
the line, exhibiting slight curvature.</p></li>
</ul></li>
</ul>
<p><strong>Distribution of p-value for all pairs</strong></p>
<p><img src="assets/perturb_pval_hist.png" style="width:60.0%" /></p>
<p><img src="assets/perturb_pval_qq.png" style="width:60.0%" /></p>
<p><strong>Distribution of p-value for GWAS pairs</strong></p>
<p><img src="assets/perturb_pval_hist_gwas_genes.png"
style="width:60.0%" /></p>
<p><img src="assets/perturb_pval_qq_gwas_genes.png"
style="width:60.0%" /></p>
</div>
<div id="results" class="section level4">
<h4>Results</h4>
<ul>
<li><p>Two WES significant genes are also detected by COTA:</p>
<ul>
<li>DNMT3A, TNFRSF6B</li>
</ul></li>
</ul>
<table class="table" style="width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:right;">
DNMT3A
</th>
<th style="text-align:right;">
TNFRSF6B
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 3cm; ">
Min
</td>
<td style="text-align:right;width: 3cm; ">
0.000049
</td>
<td style="text-align:right;">
0.000122
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Max
</td>
<td style="text-align:right;width: 3cm; ">
0.999019
</td>
<td style="text-align:right;">
0.999877
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Mean
</td>
<td style="text-align:right;width: 3cm; ">
0.501003
</td>
<td style="text-align:right;">
0.489069
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Median
</td>
<td style="text-align:right;width: 3cm; ">
0.499982
</td>
<td style="text-align:right;">
0.471279
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
SD
</td>
<td style="text-align:right;width: 3cm; ">
0.295453
</td>
<td style="text-align:right;">
0.306062
</td>
</tr>
</tbody>
</table>
<ul>
<li><p>ATG4C and SBNO2 are not detected by COTA</p>
<ul>
<li>Check the trans effect of these two genes in perturb-seq data</li>
</ul></li>
</ul>
<table class="table" style="width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:right;">
ATG4C
</th>
<th style="text-align:right;">
SBNO2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 3cm; ">
Min
</td>
<td style="text-align:right;width: 3cm; ">
0.000533
</td>
<td style="text-align:right;">
0.000260
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Max
</td>
<td style="text-align:right;width: 3cm; ">
0.999828
</td>
<td style="text-align:right;">
0.997488
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Mean
</td>
<td style="text-align:right;width: 3cm; ">
0.491614
</td>
<td style="text-align:right;">
0.507823
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Median
</td>
<td style="text-align:right;width: 3cm; ">
0.489878
</td>
<td style="text-align:right;">
0.492926
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
SD
</td>
<td style="text-align:right;width: 3cm; ">
0.292780
</td>
<td style="text-align:right;">
0.296452
</td>
</tr>
</tbody>
</table>
<ul>
<li><p>CARD9 is again detected by COTA</p></li>
<li><p>Network</p></li>
</ul>
<p><img src="assets/network_perturb.png" style="width:100.0%" /></p>
</div>
</div>
</div>
<div id="week-8-feb-24---mar-2" class="section level2">
<h2>Week 8: Feb 24 - Mar 2</h2>
<div id="annotating-the-networks" class="section level3">
<h3>Annotating the networks</h3>
<p>After having the networks, the following question is how confident
about our result and how to interpret genes identified. To explore this,
we annotate these networks as following:</p>
<ul>
<li><p>GWAS Significance:</p>
<ul>
<li><p>Nodes are colored based on GWAS p-values.</p></li>
<li><p>Each gene is assigned the smallest p-value from nearby
SNP.</p></li>
<li><p>For regions with multiple genes, the smallest p-value of genes is
used.</p></li>
</ul></li>
<li><p>Open Targets Genetics (OTG):</p>
<ul>
<li><p>Over 5000 genes are associated with CD, but many have low
Association Scores.</p></li>
<li><p>A threshold of 0.3 was applied, selecting 159 Open Target
genes.</p></li>
<li><p>Gene names are colored blue.</p></li>
</ul></li>
<li><p>WES Genes:</p>
<ul>
<li><p>Gene names are colored red.</p></li>
<li><p>Genes present in both Open Targets and WES datasets are colored
purple.</p></li>
</ul></li>
</ul>
<p>The following are annotated network from perturb-seq data:</p>
<p><img src="assets/network_perturb_annotated.png"
style="width:100.0%" /></p>
<p>Key findings:</p>
<p>A lot of hub are not GWAS significant genes, while peripheral genes
are more likely GWAS significant genes. An explanation for this findings
is that core genes tend not to be common variants, having more selection
pressure.</p>
</div>
</div>
<div id="week-9-10-mar-3---mar-16" class="section level2">
<h2>Week 9-10: Mar 3 - Mar 16</h2>
<p>One of the goal of this project is really to interpret and explain
the GWAS findings of IBD. A simple way to do this is to summarize how
many GWAS significant loci are core or peripheral genes. We summarize
COTA result as the following:</p>
<p><strong>COTA Genes Detected</strong></p>
<table>
<colgroup>
<col width="16%" />
<col width="13%" />
<col width="17%" />
<col width="10%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"></th>
<th align="right">COTA peripheral genes</th>
<th align="right">COTA master peripheral genes</th>
<th align="right">COTA core genes</th>
<th align="right">Total number of genes explained in COTA network
(peripheral + core)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right"><strong>SNP Based</strong></td>
<td align="right">5</td>
<td align="right">7</td>
<td align="right">5</td>
<td align="right">17</td>
</tr>
<tr class="even">
<td align="right"><strong>Gene Based</strong></td>
<td align="right">7</td>
<td align="right">0</td>
<td align="right">7</td>
<td align="right">14</td>
</tr>
<tr class="odd">
<td align="right"><strong>Perturb Seq</strong></td>
<td align="right">64</td>
<td align="right">18</td>
<td align="right">24</td>
<td align="right">106</td>
</tr>
<tr class="even">
<td align="right"><strong>Three Analysis (Unique Genes)</strong></td>
<td align="right">76</td>
<td align="right">25</td>
<td align="right">31 (4 genes overlapped in three analysis)</td>
<td align="right">132</td>
</tr>
</tbody>
</table>
<ul>
<li>Number of peripheral genes (regulate 1 core gene), number of master
peripheral genes (regulate &gt; 1 core genes), and number of core genes
by using GWAS nearest genes</li>
</ul>
<p><strong>GWAS Nearest Genes</strong></p>
<table style="width:100%;">
<colgroup>
<col width="15%" />
<col width="13%" />
<col width="17%" />
<col width="10%" />
<col width="42%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"></th>
<th align="right">COTA peripheral genes</th>
<th align="right">COTA master peripheral genes</th>
<th align="right">COTA core genes</th>
<th align="right">Total number of genes explained in COTA network
(peripheral + core)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right"><strong>SNP Based</strong></td>
<td align="right">1</td>
<td align="right">6</td>
<td align="right">3</td>
<td align="right">10</td>
</tr>
<tr class="even">
<td align="right"><strong>Gene Based</strong></td>
<td align="right">2</td>
<td align="right">0</td>
<td align="right">2</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="right"><strong>Perturb Seq</strong></td>
<td align="right">14</td>
<td align="right">1</td>
<td align="right">4</td>
<td align="right">19</td>
</tr>
<tr class="even">
<td align="right"><strong>Three Analysis (Unique Genes)</strong></td>
<td align="right">17</td>
<td align="right">7</td>
<td align="right">7 (2 genes overlapped in three analysis)</td>
<td align="right">31</td>
</tr>
</tbody>
</table>
<ul>
<li>Number of peripheral genes (regulate 1 core gene), number of master
peripheral genes (regulate &gt; 1 core genes), and number of core genes
by using 1MB of the GWAS loci</li>
</ul>
<p><strong>1Mb GWAS Significant Loci</strong></p>
<table style="width:100%;">
<colgroup>
<col width="15%" />
<col width="13%" />
<col width="17%" />
<col width="10%" />
<col width="42%" />
</colgroup>
<thead>
<tr class="header">
<th align="right"></th>
<th align="right">COTA peripheral genes</th>
<th align="right">COTA master peripheral genes</th>
<th align="right">COTA core genes</th>
<th align="right">Total number of genes explained in COTA network
(peripheral + core)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right"><strong>SNP Based</strong></td>
<td align="right">4</td>
<td align="right">6</td>
<td align="right">3</td>
<td align="right">13</td>
</tr>
<tr class="even">
<td align="right"><strong>Gene Based</strong></td>
<td align="right">4</td>
<td align="right">0</td>
<td align="right">3</td>
<td align="right">7</td>
</tr>
<tr class="odd">
<td align="right"><strong>Perturb Seq</strong></td>
<td align="right">31</td>
<td align="right">3</td>
<td align="right">6</td>
<td align="right">40</td>
</tr>
<tr class="even">
<td align="right"><strong>Three Analysis (Unique Genes)</strong></td>
<td align="right">39</td>
<td align="right">9</td>
<td align="right">9 (2 genes overlapped in three analysis)</td>
<td align="right">57</td>
</tr>
</tbody>
</table>
</div>
<div id="sept-analysis-on-relevance-of-k562-to-ibd"
class="section level2">
<h2>Sept: Analysis on relevance of K562 to IBD</h2>
<p>One of the issue of K562 perturb sep data is how relevant it is to
IBD since K562 are red cell blood cell line. Couple of analysis are done
to assess the relavence of K562 on IBD.</p>
<div id="cell-type-annotation-of-k562-data" class="section level4">
<h4>Cell type annotation of K562 data</h4>
<p>One way is to annotate each cell in K562 data. The UMAP of K562
expression data looks like the following:</p>
<p><img src="assets/k562_expr_umap.png" style="width:100.0%" /></p>
<p>From the UMAP, it seems all expression are clustered together. There
is no obvious clusters from UMAP, this may suggest K562 cells are more
likely to be genetically identical.</p>
<p>To be more accurate on cell type annotation, we use scmap (R package)
to annotate K562 data based on a reference data (<a
href="https://cellxgene.cziscience.com/collections/dde06e0f-ab3b-46be-96a2-a8082383c4a1">1k1k</a>),
which is a single cell data for autoimmune disease. The overall
procedure of <strong><em>scmapCell</em></strong> based on comparing
similarities (i.e. difference of experession of each cell to mean
expression of each subclusters) of projection cells to subclusters
analyzed from reference data.</p>
<ul>
<li>In progress: require a lot memory and time.</li>
</ul>
</div>
<div id="p-value-trend-comparison-of-k562-data" class="section level4">
<h4>P-value trend comparison of K562 data</h4>
<p>Another analysis is to compare the pvalue from K562 perturb seq
analysis to eQTL (whole blood) data and Jurkat T perturb seq data. This
is to check whether trans pvalue have similar trend.</p>
<ul>
<li>K562 vs. DGN (whole blood eQTL)</li>
</ul>
<p><img src="assets/dgn_k562_pvalue.png" style="width:100.0%" /></p>
<ul>
<li>K562 vs. Jurkat T (human T-lymphocyte cell line derived from T-cell
leukemia)</li>
</ul>
<p><img src="assets/jurkat_k562_pvalue.png" style="width:100.0%" /></p>
<p>Jurkat T (human T-lymphocyte cell line derived from T-cell leukemia)
might be more relavent to IBD.</p>
<p>Compared to DGN data, there are more pvalues lines in the dotted
line, that pvalue for two datasets are the same. However the range of
pvalue are quite different for Jurkat and K562, which might due to
sample size.</p>
<p>The trend of pvalue in Jurkat and K562 are different.</p>
</div>
</div>
<div id="oct-analysis-on-other-perturb-seq-data" class="section level2">
<h2>Oct: Analysis on other Perturb-seq data</h2>
<div id="nadig-jurkat-t-cell-perturb-seq" class="section level3">
<h3>Nadig Jurkat T cell perturb seq</h3>
<p>Nadig Jurkat T cell:</p>
<ul>
<li>260487 cells</li>
<li>8588 target genes, essential genes</li>
<li>2267 guides</li>
</ul>
<p>QC steps:</p>
<ul>
<li>Remove cells without sgRNA</li>
<li>Keep sgRNA carried by more than 20 cells</li>
<li>Remove cells with mito percent &gt; 20</li>
<li>Remove chrX/Y, chrM</li>
</ul>
<p>Visulization of results</p>
<ul>
<li>Negative Control of pvalue</li>
</ul>
<p><img src="assets/nadig_sceptre_negative_control.png"
style="width:100.0%" /></p>
<ul>
<li>QQ-Plot of pvalue</li>
</ul>
<p><img src="assets/qqplot_nadig_jurkat_essential_genes.png"
style="width:100.0%" /></p>
<ul>
<li>Histogram of pvalue</li>
</ul>
<p><img src="assets/nadig_sceptre_hist_pval.png"
style="width:100.0%" /></p>
<ul>
<li>Number of pvalue under threshold</li>
</ul>
<table>
<thead>
<tr class="header">
<th>threshold</th>
<th align="right">count</th>
<th align="right">proportion</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0.05</td>
<td align="right">2,769,547</td>
<td align="right">0.155</td>
</tr>
<tr class="even">
<td>0.01</td>
<td align="right">1,434,103</td>
<td align="right">0.0804</td>
</tr>
<tr class="odd">
<td>0.001</td>
<td align="right">729,389</td>
<td align="right">0.0409</td>
</tr>
<tr class="even">
<td>0.0001</td>
<td align="right">444,594</td>
<td align="right">0.0249</td>
</tr>
<tr class="odd">
<td>0.00001</td>
<td align="right">299,523</td>
<td align="right">0.0168</td>
</tr>
</tbody>
</table>
</div>
<div id="dandelion-on-nadig-jurkat-t-perturb-seq"
class="section level3">
<h3>Dandelion on Nadig Jurkat T perturb seq</h3>
<p>Fisrt, check the overlap of burdent test genes and response genes
from perturb seq</p>
<p><img src="assets/burden_test_perturb_target_overlap.png"
style="width:100.0%" /></p>
<p>Then, performing Dandelion on trans effect between genes by using
Sceptre</p>
<ul>
<li>Peripheral genes overlapped with GWAS nearest genes as candidate
peripheral genes</li>
</ul>
<p><img
src="assets/network_annotated_gwas_overlap_jurkat_essential_genes.png"
style="width:100.0%" /> - <strong>Disease Proximal Genes
(DPGs)</strong>: IL10RB, DNMT3A, ATG4C, TSPAN17, SF3B1, STRADB, USP36,
LCP2 - ATG4C, and DNMT3A are burdent test significant genes - LCP2,
ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes, regulating by
multiple peripheral genes</p>
<ul>
<li>All peripheral genes as candidate peripheral genes</li>
</ul>
<p><img src="assets/network_annotated_all_jurkat_essential_genes.png"
style="width:100.0%" /></p>
<ul>
<li><strong>Disease Proximal Genes (DPGs)</strong>: IL10RB, DNMT3A,
ATG4C, TSPAN17, SF3B1, STRADB, USP36, LCP2, GNAI3, ING1</li>
<li>ATG4C, and DNMT3A are burdent test significant genes</li>
<li>LCP2, ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes,
regulating by multiple peripheral genes</li>
</ul>
<div id="repeat-all-dandelion-analysis-on-it-with-mito-percent"
class="section level4">
<h4>Repeat all Dandelion analysis on it with mito percent</h4>
<p>Fisrt, the correlation between 2 result</p>
<p><img src="correlation_sceptre_mito_percent.png"
style="width:100.0%" /></p>
<p>Then, performing Dandelion on trans effect between genes by using
Sceptre</p>
<ul>
<li>Peripheral genes overlapped with GWAS nearest genes as candidate
peripheral genes</li>
</ul>
<p><embed
src="assets/network_annotated_gwas_overlap_jurkat_essential_genes_mito_percent.pdf"
style="width:100.0%" /> - <strong>Disease Proximal Genes
(DPGs)</strong>: IL10RB, DNMT3A, ATG4C, TSPAN17, SF3B1, STRADB, USP36,
LCP2 - ATG4C, and DNMT3A are burdent test significant genes - LCP2,
ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes, regulating by
multiple peripheral genes</p>
<ul>
<li>All peripheral genes as candidate peripheral genes</li>
</ul>
<p><embed
src="assets/network_annotated_all_jurkat_essential_genes_mito_percent.pdf"
style="width:100.0%" /></p>
<ul>
<li>GNAI3 is missing from this</li>
</ul>
</div>
</div>
<div id="cross-mappability-analysis" class="section level3">
<h3>Cross-mappability analysis</h3>
<p><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6305209/">Cross
Mappability</a> for gene A to gene B (crossmap(A,B), defined as the
number of gene A’s k-mers (75-mers from exons and 36-mers from UTRs)
whose alignment start within exonic or untranslated regions of gene B.
Only genes with mappability &lt; 1 (not NA) appear in cross mapping
files. In other words, it is the count of how many k-mers from A map to
B.</p>
<p>Because crossmap(A, B) <span class="math inline">\(\ne\)</span>
crossmap(B, A) (A may have longer exons or more repetitive sequence), a
symmetric version is defined <span
class="math inline">\(\frac{\text{crossmap(A,B)} +
\text{crossmap(B,A)}}{2}\)</span>.</p>
<p>The range for cross mappability is larger than 0. And the summary for
cross mappability as the following</p>
<ul>
<li>Distribution of cross mappability</li>
</ul>
<p><img src="assets/hist_cross_mappability.png"
style="width:100.0%" /></p>
<p><img src="assets/box_plot_cross_mappability.png"
style="width:100.0%" /></p>
<table>
<thead>
<tr class="header">
<th align="center">Percentile</th>
<th align="center">Cross Mappability</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">0%</td>
<td align="center">1</td>
</tr>
<tr class="even">
<td align="center">10%</td>
<td align="center">2</td>
</tr>
<tr class="odd">
<td align="center">20%</td>
<td align="center">4</td>
</tr>
<tr class="even">
<td align="center">30%</td>
<td align="center">7</td>
</tr>
<tr class="odd">
<td align="center">40%</td>
<td align="center">10</td>
</tr>
<tr class="even">
<td align="center">50%</td>
<td align="center">14</td>
</tr>
<tr class="odd">
<td align="center">60%</td>
<td align="center">19</td>
</tr>
<tr class="even">
<td align="center">70%</td>
<td align="center">26</td>
</tr>
<tr class="odd">
<td align="center">80%</td>
<td align="center">36</td>
</tr>
<tr class="even">
<td align="center">90%</td>
<td align="center">58</td>
</tr>
<tr class="odd">
<td align="center">100%</td>
<td align="center">16921</td>
</tr>
</tbody>
</table>
<p>Most of the cross-mappability is less than 30. It is heavy tail, a
lot of outliers.</p>
<ul>
<li>Distribution of symmetric cross mappability</li>
</ul>
<p><img src="assets/hist_symmetric_cross_mappability.png"
style="width:100.0%" /></p>
<p><img src="assets/box_plot_symmetric_cross_mappability.png"
style="width:100.0%" /></p>
<table>
<thead>
<tr class="header">
<th align="center">Percentile</th>
<th align="center">Symmetric Cross Mappability</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">0%</td>
<td align="center">0.5</td>
</tr>
<tr class="even">
<td align="center">10%</td>
<td align="center">1.0</td>
</tr>
<tr class="odd">
<td align="center">20%</td>
<td align="center">2.5</td>
</tr>
<tr class="even">
<td align="center">30%</td>
<td align="center">4.0</td>
</tr>
<tr class="odd">
<td align="center">40%</td>
<td align="center">6.0</td>
</tr>
<tr class="even">
<td align="center">50%</td>
<td align="center">8.0</td>
</tr>
<tr class="odd">
<td align="center">60%</td>
<td align="center">11.5</td>
</tr>
<tr class="even">
<td align="center">70%</td>
<td align="center">16.0</td>
</tr>
<tr class="odd">
<td align="center">80%</td>
<td align="center">23.0</td>
</tr>
<tr class="even">
<td align="center">90%</td>
<td align="center">38.5</td>
</tr>
<tr class="odd">
<td align="center">100%</td>
<td align="center">15660.5</td>
</tr>
</tbody>
</table>
<p>Similarly, most of the cross-mappability is less than 30. It is heavy
tail, a lot of outliers.</p>
<ul>
<li><p>Dicussion: how to set threshold to define high cross
mappability</p>
<ul>
<li>Normalization: require length of gene A ?</li>
<li>By distribution ?</li>
</ul></li>
</ul>
</div>
</div>
<div id="qa-on-the-project" class="section level2">
<h2>Q&amp;A on the Project</h2>
<ol style="list-style-type: decimal">
<li>Why just burden test</li>
</ol>
<ul>
<li>Can be flexible to many gene effect methods</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Why not leverage cis effect</li>
</ol>
<ul>
<li>Cis effect may not be detected due to stronger selective
constraints.</li>
</ul>
<div id="next-steps" class="section level3">
<h3>Next Steps</h3>
<ol style="list-style-type: decimal">
<li><strong>Map GWAS Loci</strong>
<ul>
<li>Utilize <strong>SNPeff</strong> and
<strong>Fine-mapping</strong></li>
<li>Apply <strong>ABC method</strong> (<a
href="https://www.nature.com/articles/s41586-024-07022-x">Nature
Link</a>)</li>
</ul></li>
<li><strong>Perform Perturb-seq for trans-effects</strong>
<ul>
<li>Utilize <strong>CRISPRi</strong></li>
<li>Direct application in <strong>COTA analysis</strong>.</li>
</ul></li>
</ol>
</div>
</div>
<div id="project-overview" class="section level2">
<h2>Project Overview</h2>
<ol style="list-style-type: decimal">
<li>Fine-mapping of GWAS loci</li>
</ol>
<ul>
<li>Find casual gene for each loci</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Connect core gene to the network</li>
</ol>
<ul>
<li>Gene expressed in certain cell type</li>
<li>trans-regulation path also expressed in certain cell type</li>
</ul>
<ol start="4" style="list-style-type: decimal">
<li>Interpret GWAS loci not related to the core genes</li>
</ol>
<ul>
<li>pathway</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Next level: Difference in UC and CD</li>
</ol>
<ul>
<li>Shared</li>
<li>Unique</li>
</ul>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.4.3 (2025-02-28)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] kableExtra_1.4.0 knitr_1.50      

loaded via a namespace (and not attached):
 [1] jsonlite_2.0.0     compiler_4.4.3     promises_1.3.3     Rcpp_1.0.14       
 [5] xml2_1.3.8         stringr_1.5.1      git2r_0.36.2       later_1.4.2       
 [9] jquerylib_0.1.4    textshaping_1.0.1  systemfonts_1.2.3  scales_1.4.0      
[13] yaml_2.3.10        fastmap_1.2.0      R6_2.6.1           workflowr_1.7.1   
[17] tibble_3.2.1       rprojroot_2.0.4    svglite_2.2.1      bslib_0.9.0       
[21] pillar_1.10.2      RColorBrewer_1.1-3 rlang_1.1.6        cachem_1.1.0      
[25] stringi_1.8.7      httpuv_1.6.16      xfun_0.52          fs_1.6.6          
[29] sass_0.4.10        viridisLite_0.4.2  cli_3.6.5          magrittr_2.0.3    
[33] digest_0.6.37      rstudioapi_0.17.1  lifecycle_1.0.4    vctrs_0.6.5       
[37] evaluate_1.0.3     glue_1.8.0         farver_2.1.2       whisker_0.4.1     
[41] rmarkdown_2.29     tools_4.4.3        pkgconfig_2.0.3    htmltools_0.5.8.1 </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
