<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>rotation.knit</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Research Progress</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Progress
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="2025.html">2025</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/zq2209/GradPLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-06-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradPLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220610code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220610code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220610)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtree7afa9e6fabf31b3496fe38d4bfd2d70dc4ad3768targetblank7afa9e6a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/zq2209/GradPLog/tree/7afa9e6fabf31b3496fe38d4bfd2d70dc4ad3768" target="_blank">7afa9e6</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomzq2209GradPLogtree7afa9e6fabf31b3496fe38d4bfd2d70dc4ad3768targetblank7afa9e6a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/zq2209/GradPLog/tree/7afa9e6fabf31b3496fe38d4bfd2d70dc4ad3768" target="_blank">7afa9e6</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
working directory clean
</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/rotation.Rmd</code>) and HTML
(<code>docs/rotation.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/a0a1f620dde561cb6b8e81a5d5af54ec47d1e071/analysis/rotation.Rmd" target="_blank">a0a1f62</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/a0a1f620dde561cb6b8e81a5d5af54ec47d1e071/docs/rotation.html" target="_blank">a0a1f62</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/ae3d616baf466bf969c8f7ce9d932b1bb26f5712/analysis/rotation.Rmd" target="_blank">ae3d616</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/ae3d616baf466bf969c8f7ce9d932b1bb26f5712/docs/rotation.html" target="_blank">ae3d616</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/1d71192b97903efe91fd43d6f192f8cce4b8fc6d/analysis/rotation.Rmd" target="_blank">1d71192</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/bd405e1f3a1be3e0d0df6a654c6e2ab98bebb526/analysis/rotation.Rmd" target="_blank">bd405e1</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/bd405e1f3a1be3e0d0df6a654c6e2ab98bebb526/docs/rotation.html" target="_blank">bd405e1</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/509f6cafb87dd2491906712eb584b5404ffcbe66/analysis/rotation.Rmd" target="_blank">509f6ca</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/zq2209/GradPLog/509f6cafb87dd2491906712eb584b5404ffcbe66/docs/rotation.html" target="_blank">509f6ca</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
update rotation page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/zq2209/GradPLog/blob/783dfdbfcd939c5be689b5cdfd47ca9aee464d60/analysis/rotation.Rmd" target="_blank">783dfdb</a>
</td>
<td>
zq2209
</td>
<td>
2025-06-25
</td>
<td>
add ppi rotation project
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="goal" class="section level2">
<h2>Goal</h2>
<ul>
<li><p>Investigate potential mechanisms that may underlie genetic
variants that affect a gene’s protein expression level in trans, but not
its mRNA expression level.</p></li>
<li><p>Curate databases that capture different mechanisms including
annotated strict protein complex partners, many different methods that
identify protein-protein interaction networks, known signaling pathway,
and more.</p></li>
<li><p>Develop approaches that may help distinguish trans-pQTLs that are
driven by cell-composition effects versus intra-cellular trans-pQTL
effects</p></li>
</ul>
</div>
<div id="test-whether-pqtl-enriched-in-protein-pairs"
class="section level2">
<h2>Test whether pQTL enriched in protein pairs</h2>
<div id="dataset" class="section level3">
<h3>Dataset</h3>
<ul>
<li><p>PPI</p>
<ul>
<li><p>Bioplex: ~ 2900 pairs after overlapping with pQTL data</p></li>
<li><p>Hippie: ~ 1346 pairs after overlapping</p></li>
<li><p>Corum: ~ 1270 pairs after overlapping</p></li>
<li><p>String: ~ 6750 pairs after overlapping</p></li>
</ul></li>
<li><p>Gene module</p>
<ul>
<li><p>DNG gene module by trans-PCO:</p>
<ul>
<li><p>166 gene modules</p></li>
<li><p>12000+ genes in total</p></li>
<li><p>over 1,000,000 gene pairs</p></li>
</ul></li>
</ul></li>
<li><p>QTL summary statistics</p>
<ul>
<li><p>UKB-PPP: plasma</p></li>
<li><p>DGN: whole blood</p></li>
</ul></li>
</ul>
</div>
<div id="prodecure" class="section level3">
<h3>Prodecure</h3>
<ol style="list-style-type: decimal">
<li><p>For each gene/protein pair, pick up the SNP from the most
significant transQTL (i.e. smallest pvalue) of gene/protein A.</p></li>
<li><p>For gene/protein B of this gene pair, pick up pvalue of QTL for
this SNP and B.</p></li>
<li><p>After fisrt 2 steps, there will be a pair of pvalues, then
calculate the proportion that both pvalues are significant under certain
pvalue threshold, i.e. the probability of there is transQTL for B given
there is transQTL for A.</p></li>
<li><p>Use bootstrap to somehow mimic the distribution of the
probability. For each dataset, each time select certain number of pairs
and calculate probability of this dataset. From boostrap, there will be
distribution of probability, able to estimate standard errors and
construct confidence intervals.</p></li>
</ol>
<p>Note: Not sure about which pvalue threshold and sample size of
bootstrap, so have results for multiple pvalue thresholds and boostrap
sample sizes.</p>
</div>
<div id="preliminary-result" class="section level3">
<h3>Preliminary result</h3>
<ul>
<li>Plot for probability of there is transQTL for B given there is
transQTL for A with bootstrap</li>
</ul>
<p><img src="assets/ppi_v1.png" style="width:100.0%" /></p>
<p>The overall trend as expected, PPI pairs have higher probability then
random pairs. And the error bar decrease as sample size of bootstrap
increase.</p>
<ul>
<li>Table for cisQTL for B given there is transQTL for A</li>
</ul>
<table class="table" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>
Table: Total Number of Cis-Trans Pairs
</caption>
<thead>
<tr>
<th style="text-align:center;">
Dataset
</th>
<th style="text-align:center;">
Total
</th>
<th style="text-align:center;">
trans.cis
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;width: 5cm; ">
bioplex
</td>
<td style="text-align:center;width: 4cm; ">
5800
</td>
<td style="text-align:center;width: 4cm; ">
21
</td>
</tr>
<tr>
<td style="text-align:center;width: 5cm; ">
corum
</td>
<td style="text-align:center;width: 4cm; ">
2542
</td>
<td style="text-align:center;width: 4cm; ">
25
</td>
</tr>
<tr>
<td style="text-align:center;width: 5cm; ">
hippie
</td>
<td style="text-align:center;width: 4cm; ">
2692
</td>
<td style="text-align:center;width: 4cm; ">
11
</td>
</tr>
<tr>
<td style="text-align:center;width: 5cm; ">
random
</td>
<td style="text-align:center;width: 4cm; ">
4471
</td>
<td style="text-align:center;width: 4cm; ">
6
</td>
</tr>
</tbody>
</table>
<p>Note: Total pairs is the number of unique pairs * 2</p>
</div>
<div id="consideration-disscussion" class="section level3">
<h3>Consideration &amp; Disscussion</h3>
<ol style="list-style-type: decimal">
<li>The tissue difference may have impact on the result</li>
</ol>
<ul>
<li><p>Different tissue of PPI datasets:, e.g whole blood cells for
bioplex and general cells for others</p></li>
<li><p>Different tissue in pQTL and eQTL data</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>The technology difference may also have some impact</li>
</ol>
<ul>
<li>e.g. bioplex test for interaction instead of complex</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>The difference in number of pairs</li>
</ol>
</div>
<div id="next-step" class="section level3">
<h3>Next step</h3>
<ol style="list-style-type: decimal">
<li><p>Summarize the gene pair result</p></li>
<li><p>Probability of transQTL of B given cisQTL for A, and any other
possible combination</p></li>
<li><p>Any other PPI module</p></li>
</ol>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.4.3 (2025-02-28)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] kableExtra_1.4.0 knitr_1.50      

loaded via a namespace (and not attached):
 [1] jsonlite_2.0.0     compiler_4.4.3     promises_1.3.3     Rcpp_1.0.14       
 [5] xml2_1.3.8         stringr_1.5.1      git2r_0.36.2       later_1.4.2       
 [9] jquerylib_0.1.4    textshaping_1.0.1  systemfonts_1.2.3  scales_1.4.0      
[13] yaml_2.3.10        fastmap_1.2.0      R6_2.6.1           workflowr_1.7.1   
[17] tibble_3.2.1       rprojroot_2.0.4    svglite_2.2.1      bslib_0.9.0       
[21] pillar_1.10.2      RColorBrewer_1.1-3 rlang_1.1.6        cachem_1.1.0      
[25] stringi_1.8.7      httpuv_1.6.16      xfun_0.52          fs_1.6.6          
[29] sass_0.4.10        viridisLite_0.4.2  cli_3.6.5          magrittr_2.0.3    
[33] digest_0.6.37      rstudioapi_0.17.1  lifecycle_1.0.4    vctrs_0.6.5       
[37] evaluate_1.0.3     glue_1.8.0         farver_2.1.2       whisker_0.4.1     
[41] rmarkdown_2.29     tools_4.4.3        pkgconfig_2.0.3    htmltools_0.5.8.1 </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
