---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```

## Week 1: Jan 6 - Jan 12

### Current Research on IBD

This section summarize the WES and GWAS studies on IBD, giving background information on the disease and current research on the diease. 

#### Disease Background

##### General Background of IBD

![](assets/image1.png){width="60%"}

- IBD is Chronic inflammation influenced by genetics, environment, microbiota, and immunity.

- Genetic Contribution

  - Crohn's Disease (CD): 15% family history; twin studies show 50% concordance in monozygotic (MZ) twins vs. less than 10% in dizygotic (DZ) twins.

  - GWAS identified 163 loci; trans-ethnic studies identified an additional 38 loci.

- Epigenetics

  - Genome-environment interactions affect disease progression.

  - Emerging research focuses on the role of epigenetics in IBD.


##### Genetic Associations in IBD

- NOD2: First CD-associated gene (2001), with key variants R702W and G908R.

- Autophagy Genes: ATG16L1, LRRK2, and IRGM predispose individuals to IBD.

- IL-10 Receptor Mutations: IL10RA, IL10RB are linked to colitis.

- IBD-Associated Loci: ~240 loci identified (as of 2022); 30 shared between CD and UC.

- CD Predictive Loci: FOXO3, IGFBP1, and XACT as potential markers.

#### Large-Scale Exome Sequencing Identifies Novel Genes and Pathways in Crohn’s Disease

Sazonovs, Aleksejs et al.

##### Motivation

- CD is a chronic inflammatory disorder with a strong genetic component.

- GWAS has primarily focused on common variants, but rare coding variants remain under-explored.

##### Study Design

![](assets/image2.png){width="60%"}

##### Findings
- Single Variant Analysis: 94 out of 116 variants replicated in discovery datasets.

![](assets/image3.png){width="80%"}
![](assets/image4.png){width="60%"}


- Burden Test: Missense vs. frameshift variants analyzed.

![](assets/image5.png){width="90%"}


##### Takeaways

- Exome Sequencing Complements GWAS
  - Addresses gaps in genetic architecture (low-frequency and rare variants) that earlier CD GWAS
  meta-analyses could not observe.

- Significance of Coding Variants:
  - Coding variants, although fewer than noncoding variants, are highly enriched for associations with
  both common and rare diseases.
  - Tend to have stronger effects due to natural selection, which often keeps their frequencies low.
  - Provide direct links to specific genes and pathogenic mechanisms, unlike noncoding variants.

- Key Findings in CD Pathogenesis:
  - Novel genes such as PDLIM5, SDF2L1, HGFAC, PAF-R, and CCR7 emphasize the role of mesenchymal cells
  (MCs) in intestinal inflammation.

- Therapeutic Implications:
  - Findings highlight the potential for therapeutic strategies targeting mesenchymal niche balance to
  address CD pathogenesis.
  - Genetic evidence for drug targets is valuable for driving drug development.
  
- Future Directions:
  - Expanded sequencing efforts in ulcerative colitis and integrated analyses with larger GWAS studies
  are expected to identify more linked genes.

#### Genome-wide association study implicates immune activation of multiple integrin genes in inflammatory bowel disease

de Lange, K., Moutsianas, L., Lee, J. et al.

##### Motivation

- Current treatments involve immunomodulators, but patients often experience side effects or treatment resistance.

- GWAS and Immunochip studies have identified risk loci but have had limited therapeutic impact.

##### Study Design

![](assets/image6.png){width="90%"}


##### Findings
Identified 25 new GWAS loci

- 4 loci with significant variants
  - SLAMF8
  - RORC
  - PLCG2
  - NCF4
  
- Another 4 loci within integrin gene
  - 3 loci showed larger than 90% probability have colocalization with eQTL
  - 1 loci showed intermediate evidendence
  

##### Takeaways

- Integrins are not only important in cell trafficking but can also participate in cellular signaling.

- Highlighted integrins as key therapeutic targets:
  - Monoclonal antibodies like vedolizumab and etrolizumab targeting integrins have shown
  efficacy in IBD.
  - Identified SMAD7, a modulator of $TGF-\beta$ signaling, as a potential target for Crohn’s disease
  treatment.

- Emphasized the importance of gut-selective therapies to minimize risks like progressive multifocal
leukoencephalopathy (PML).

- These discoveries have demonstrated that the effect sizes of GWAS associations do not necessarily
reflect the importance or therapeutic relevance of their underlying biological pathways.


### Method

#### Prioritizing disease-mediating genes leveraging trans-gene regulation

##### Motivation

- Complex Genetic Architecture of Diseases:
  - Complex traits are highly polygenic, involving numerous genes with varying effect sizes.
  - The omnigenic model posits that all expressed genes in disease-relevant tissues contribute to disease
  risk, but not all genes have equal importance.
  
- Importance of Disease-Mediating Genes:
  - Disease-associated genes can be categorized into:
    - Disease-mediating genes: Directly affect the disease.
    - Peripheral genes: Indirectly influence the disease through trans-regulatory networks.

  - Identifying disease-mediating genes is critical for uncovering pathways and mechanisms relevant to
  treatment development.

- Limitations of Current Gene Prioritization Methods:
  - GWAS:
    - Identifies important disease-mediating genes but may miss many due to lack of power in detecting
    low-frequency or rare variants with large effects.

  - Rare Variant Methods:
    - Gene-based approaches, like burden tests in WES, improve detection but are limited by current
    sample sizes.
    
  - Graph and Canonical Correlation Models:
    - Recent approaches predict candidate genes or link expressions to variants but fail to explicitly
    differentiate disease-mediating from peripheral genes or reveal trans-regulatory networks.

- Challenges with Cis-eQTLs in Disease Gene Discovery:
  - Disease genes often lack strong cis-QTL signals due to stronger selective constraints.
  - Cis-eQTL limitations necessitate focusing on trans-QTLs, as their target genes share similar
  selective constraints with disease genes.

- Significance of Trans-Regulation:
  - Trans-regulatory networks explain a large portion (~70%) of disease heritability.
  - Understanding trans-regulation provides insights into collaborative gene contributions to disease
  risk and highlights the importance of completing trans-gene regulatory networks for understanding
  complex trait genetics.

##### COTA
- COTA Model: Integrates trans-regulatory effects to identify disease-mediating genes.

- GBAT: Predicts gene expression using machine learning models.

![](assets/image7.png){width="80%"}

- DACT: Estimates gene effects from burden tests.


![](assets/image8.png){width="80%"}

##### Findings

- COTA enhances GWAS interpretability by revealing trans-regulatory networks.

- New gene discoveries provide insights into disease mechanisms.

- Potential for targeted therapy development based on genetic findings.

##### Code

- Codes for COTA are in [COTA Package.](https://github.com/mxxptian/COTA/tree/d211df91c43826ae56adfa30b39d506716921ba7)

- Summary of codes are in [code summary.](https://docs.google.com/document/d/1i0V_S7Yh-XlwvDcbUzki3WhIxmeUiaPs_Eazlr4lhys/edit?tab=t.0) 


## Week 2: Jan 13 - Jan 19

### Data
#### Whole Exon Sequencing 

##### Files
- UC: Burden tests with UC only
- CD: Burden tests with CD only
- IBD: Burden tests with other types of IBD
  
#### Variants analyzed
- NSYN: Non-synonymous Variants
  - nsyn.singleton: variants appear only once across all samples
  - nsyn.0.001: allele frequancy less than 0.001
- NSYN_AM: Non-synonymous variants with an AlphaMissense score greater than 0.2
  - nsyn.singleton: variants appear only once across all samples
  - nsyn.0.001: allele frequancy less than 0.001
- pLoF: PTV variants that are also flagged as high confidence (HC) by Loftee
  - pLoF.singleton: variants appear only once across all samples
  - pLoF.nsyn.0.001: allele frequancy less than 0.001


##### Variables 
- MarkerName: Gene name
- Allele1: Study allele
- Allele2: Ref allele
- Freq1: Allele frequency of Allele1
- Effect: Effect size
- StdErr: Standrad error of effect size
- log(P): Log p-value of effect size
- Direction: Direction of effect size from two platform 
- HetISq: Heterogeneity statistics I-square
- HetChiSq: Chi-square statistics for heterogeneity that effect for two platform are come from same distribution
- HetDf: Degree of freedom
- logHetP: Log p-value of HetChiSq
  
##### EDA of data
- Basic visualizations of **p-values** were performed, including **QQ plots** and **histograms**, for burden tests involving **singleton** and **<0.001 NSYN/LoF variants** in **Crohn’s Disease (CD)**.  
  - The overall distribution of **p-values** for **CD with NSYN and LoF variants** appears nearly identical.  
  - Using a uniform distribution, the points predominantly align with the line, exhibiting slight curvature.
  
![](assets/image10.png){width="50%"}

![](assets/image11.png){width="50%"}

- Significant heterogeneity across all **nine studies** was assessed.  
  - Two genes met the significance threshold of **0.05/20,000** (**total genes: 18,641**):  
    - **SULT1A1 (Sulfotransferase Family 1A Member 1):** Encodes sulfotransferase enzymes that facilitate
    **sulfate conjugation** of various hormones, neurotransmitters, drugs, and xenobiotic compounds.
    These cytosolic enzymes differ in **tissue distribution** and **substrate specificity**.  
    - **PLXND1 (Plexin D1):** Enables **protein domain-specific binding activity** and is involved in
    **axonogenesis regulation**, **semaphorin-plexin signaling**, and **synapse assembly**. Also plays a
    role in **circulatory system development**, **salivary gland branching**, and **kidney development**,
    and is primarily located in **lamellipodia**.  
  - The paper reported **43 sites** with significant heterogeneity, whereas only **two genes** were
  identified in this analysis.  
  - The original study applied a **heterogeneity-of-effect test** between the **Nextera** and **Twist
  discovery cohorts** using **IVW PHET < 0.0001** as the threshold.  
  - The discrepancy may be due to **differences in heterogeneity statistics** and a **more lenient
  p-value cutoff** in this analysis.  
  - When **0.0001** was used as the threshold, **32 genes** exhibited significant heterogeneity,
  including **SULT1A1 and PLXND1**.  

- **Basic summary of genes in the WES dataset**:  
  - **18,641 genes** are present in the dataset.  
  - **15,309 genes** are included in **all nine studies**.  
  - **3,332 genes** are shared across some but not all studies.  
  - In the **CD with NSYN 0.01 study**, **64 genes** lack **burden tests on singleton variants**.  
  - Some genes do not have variants that appear in only one sample.
  
![](assets/image12.png){width="80%"}

![](assets/image13.png){width="80%"}

#### Trans Effect

- trans-eQTL: 
  - eQTLGen
  - Whole blood
  
- GBAT

#### GWAS

- The dataset consists of **29,849 East Asian samples** from China, Hong Kong SAR, Japan, and the Republic of Korea, and **368,819 European samples** from the U.S., NR, and Finland.  

- The total number of **cases** in the dataset is **45,106**, while the total number of **controls** is **353,562**.  

- Among the **East Asian cohort**, there are **14,393 cases** and **15,456 controls**.  

- Among the **European cohort**, there are **30,713 cases** and **338,106 controls**.  

- The majority of cases are diagnosed with **Crohn’s Disease (CD)** and **Ulcerative Colitis (UC)**, with a smaller number of **other Inflammatory Bowel Disease (IBD) cases**.  

- The **East Asian samples represent approximately 7.5%** of the total dataset.


![](assets/image9.png){width="80%"}

## Week 3: Jan 20 - Jan 26

### Analysis

#### Data
##### WES Data
- CD with Variant Frequency less than 0.001
- Total Genes: 18,620

##### Genes Detected by WES
- Total of 32 significant genes.
- **Key Genes Identified**:
  - **NOD2**: Plays a role in **autophagy**, aiding in the destruction of bacteria, viruses, and harmful substances.
  - **SBNO2**: Differentially regulated by **lipopolysaccharide and IL-10** ([Nature Link](https://www.nature.com/articles/s41467-024-47218-3)).
  - **ATG4C**: Involved in **autophagy**, the process of degrading proteins and organelles.
  - **IGHV Family**: Immunoglobulin heavy variable genes.
  - **AHNAK2**: Encodes a **large nucleoprotein**.

#### SNP-Based Analysis
- GWAS Significant SNPs (*p* < 1e-4): 69,654 SNPs
  - Overlap with trans-eQTL SNPs: 1033 SNPs
- Total Trans-eQTL SNPs: 10,317

#### Gene-Based Analysis
- GWAS Significant Genes** (*p* < 1e-4): 69,654 SNPs
  - Overlap of GWAS Nearest Gene with GBAT trans regulation target: 2134
- Total Trans-eQTL Gene1: 13,634

## Week 4: Jan 27 - Feb 2

### Findings

#### Genes Detected by COTA (SNP-Based)
- **Significant Gene Pairs**:
  - Shown in [significant gene pairs]()
  
- **WES Significant Gene2**: NOD2, SBNO2

- **WES Non-Significant Gene2**: CARD9, CEACAM8, USP36

- **Network of detected gene**:


![](assets/network_snp.png){width="100%"}


#### Genes Detected by COTA (Gene-Based)
- **Significant Gene Pairs**:
  - Shown in [significant gene pairs]()
  
- **WES Significant Gene2**: NOD2, AHNAK2

- **WES Non-Significant Gene2**: SH3YL1, TMED6, CEP104, SPIRE2, ING1, EGLN3, PLEKHO2, EML4, ALOX5
  

- **Network of detected gene**:

![](assets/network_gene.png){width="100%"}


## Week 5: Feb 3 - Feb 9

### Perturb-seq data
How does the data generated   

- what is cell line   

- trans regulation target, gene_id? 

  - direction of correlation, which one regulate which?      
  
- overlap of burden test significant genes and target genes is larger      

  - 13407 total regulator, and all 32 burden test genes are in regulator    
  
  - 4165 of trans target, 20 burden test genes are in trans target    
  
- QTL calculation

  - why negative binomial 
    
    - Count data that over dispersed?     

  
  
## Week 6: Feb 10 - Feb 16 - Discussion

### Why most of burden tests significant genes are not detected by COTA

#### Check the overlap of burden tests significant genes and trans regulation target

- SNP based
  - 5 significant genes from the burden tests (AHNAK2, DNMT3A, NOD2, SBNO2, ATG4C) overlap with the trans-eQTL genes used for analysis. 
  
![](assets/image14.png){width="60%"}
  
- Gene based
  - 6 significant genes from the burden tests (AHNAK2, DNMT3A, NOD2, SBNO2, ATG4C) overlap with the trans regulation targets used for analysis.
  
![](assets/image15.png){width="60%"}

#### Check DGN matrix

There are still 5 significant burden test genes (AHNAK2, DNMT3A, NOD2, SBNO2, ATG4C) overlapped with DGN.

#### Check GTEx 

- Bulk gene expression have all the significant burden test genes. 
- Missed burden test significant genes 
  - immunoglobulin heavy joining
    - IGHJ4 (immunoglobulin heavy joining 4; chr14:105864215-105864260:-): Highly expressed at Cells - EBV-transformed lymphocytes and Spleen, intermediate expression in Whole blood
    
    ![](assets/IGHJ4_tpm.png){width="80%"}
    
  - immunoglobulin heavy variable
    - IGHV4-34 (immunoglobulin heavy variable 4-34; chr14:106373663-106374145:-): Highly expressed at Cells - EBV-transformed lymphocytes
    
    ![](assets/IGHV4-34_tpm.png){width="80%"}
    
  - immunoglobulin kappa joining 
    - IGKJ1 (immunoglobulin kappa joining 1; chr2:88861886-88861923:-): Highly expressed at Cells - EBV-transformed lymphocytes, intermediate expression in Spleen and Colon - Transverse, Small Intestine - Terminal Ileum, Minor Salivary Gland
    
    ![](assets/IGKJ1_tpm.png){width="80%"}
    
  - immunoglobulin kappa variable
   - IGKV4-1 (immunoglobulin kappa variable 4-1; chr2:88885397-88886153:+): Highly expressed at Cells - EBV-transformed lymphocytes
    
    ![](assets/IGKV4-1_tpm.png){width="80%"}
    
  - immunoglobulin heavy diversity
    - IGHD3-16 (immunoglobulin heavy diversity 3-16; chr14:105895634-105895670:-): Highly expressed at Lung, Cells - EBV-transformed lymphocytes, and Spleen
    
    ![](assets/IGHD3-16_tpm.png){width="80%"}
    
 - TNFRSF6B (TNF receptor superfamily member 6b; chr20:63696652-63698641:+): Highly expressed in Lung???
 
  ![](assets/TNFRSF6B_tpm.png){width="80%"}

- Overlapped burden test significant genes
  - NOD2 (nucleotide binding oligomerization domain containing 2; chr16:50693588-50734041:+): Expression in Whole Blood are higher than others
  
  ![](assets/NOD2_tpm.png){width="80%"}
  
  - ATG4C (autophagy related 4C cysteine peptidase; chr1:62784132-62865516:+): Expressed in all tissues, higher expression in brain tissue
  
  ![](assets/ATG4C_tpm.png){width="80%"}
  
  - SBNO2 (strawberry notch homolog 2; chr19:1107637-1174268:-): Expression in Whole Blood are higher than others
  
  ![](assets/SBNO2_tpm.png){width="80%"}
  
  - AHNAK2 (AHNAK nucleoprotein 2; chr14:104937762-104978374:-): Higher expression in skin tissue then whole blood
  
  ![](assets/AHNAK2_tpm.png){width="80%"}
  
  - DNMT3A (DNA methyltransferase 3 alpha; chr2:25227855-25342590:-): Expressed in all tissues, no really extream TPM values in some tissue
  
  ![](assets/DNMT3A_tpm.png){width="80%"}
  
- Check TPM of genes on bulk data
  - Near all gene expression for those missed genes in trans effect are higher, probably due to have extreme high TPM in some tissues, whereas those overlapped genes don't have such extreme expression. 
  
  ![](assets/bulk_tpm.png){width="80%"}


### Why there are many more significant genes than Sazonovs et al 2022

- The data shared to us are different than 2022, we have newer version of their results.

  - Samples are different than what used in 2022 paper, the sample size are similar but samples are different.
  
  - New exome capture method that could capture exome region previously not included. 
  
- Genes may also be not totally consistent with 2022

  - Version of gene annotation are different in two platform
  
- Immunoglobulin genes are filtered out in the analysis by thinking it as artifact

  - The samples in the consortium come from different hospitals   
  
  - There is a possibility that a small number of immortalized cell line samples may have been mixed in.  
  - These cell lines are derived from some B cells, and immunoglobulin gene region in B cells is highly recombinant.


## Week 7: Feb 17 - Feb 23

### Integrate perturb data with COTA

#### Preprocessing data

- Originally 14680 targets; 4734 regulators

- Apply two steps filter on genes

  - Filter out genes that don't have a matched gene name
  
    - drop 1536 targets and 1 regulator 
    
  - Select regulators that are GWAS gens (nearest genes of GWAS loci)
  
    - 12730 targets and 1337 regulators remain for analysis
    
- Among 12370 targets, 4 WES significant genes were included. 
  
  - DNMT3A, TNFRSF6B, ATG4C, SBNO2
  
  - NOD2 and AHNAK2 were not included in targets
  
- NAs in the p-value matrix of 12730 targets (rows) and 1337 regulators (columns)

  - These pairs (regulator - target) don't exist in the perturb-seq result
  
  - Fill in with 0.99 to make sure COTA analysis can be applied
  
  
#### Visualization of pvalue 

- Basic visualizations of **p-values** were performed, including **QQ plots** and **histograms**, for all the gene pairs and filtered gene pairs (GWAS pairs).  

  - The overall distribution of **p-values** for all pairs and GWAS pairs appears nearly identical.  
  
  - Using a uniform distribution, the points predominantly align with the line, exhibiting slight curvature.
  
**Distribution of p-value for all pairs** 

![](assets/perturb_pval_hist.png){width="60%"}

![](assets/perturb_pval_qq.png){width="60%"}

**Distribution of p-value for GWAS pairs** 

![](assets/perturb_pval_hist_gwas_genes.png){width="60%"}

![](assets/perturb_pval_qq_gwas_genes.png){width="60%"}

#### Results

- Two WES significant genes are also detected by COTA:
  
  - DNMT3A, TNFRSF6B
  
```{r, echo = FALSE}
library(knitr)
library(kableExtra)

# Create the data frame with rounded values
df <- data.frame(
  DNMT3A = round(c(0.0000490, 0.9990190, 0.5010033, 0.4999820, 0.2954534), 7),
  TNFRSF6B = round(c(0.0001217, 0.9998770, 0.4890691, 0.4712790, 0.3060624), 7),
  row.names = c("Min", "Max", "Mean", "Median", "SD")
)

# Print the table using kable
kable(df, format = "html", digits = 6) %>%
  kable_styling(full_width = FALSE) %>%
  column_spec(1, width = "3cm") %>%
  column_spec(2, width = "3cm")
```


  
- ATG4C and SBNO2 are not detected by COTA

  - Check the trans effect of these two genes in perturb-seq data 

```{r, echo = FALSE}
# Create the data frame with rounded values for the second table
df2 <- data.frame(
  ATG4C = round(c(0.0005332, 0.9998280, 0.4916141, 0.4898780, 0.2927804), 7),
  SBNO2 = round(c(0.0002595, 0.9974880, 0.5078234, 0.4929260, 0.2964521), 7),
  row.names = c("Min", "Max", "Mean", "Median", "SD")
)

# Print the second table using kable
kable(df2, format = "html", digits = 6) %>%
  kable_styling(full_width = FALSE) %>%
  column_spec(1, width = "3cm") %>%
  column_spec(2, width = "3cm")
```


- CARD9 is again detected by COTA

- Network

![](assets/network_perturb.png){width="100%"}

## Week 8: Feb 24 - Mar 2

### Annotating the networks

After having the networks, the following question is how confident about our result and how to interpret genes identified. To explore this, we annotate these networks as following:  

- GWAS Significance:
  - Nodes are colored based on GWAS p-values.
  
  - Each gene is assigned the smallest p-value from nearby SNP.

  - For regions with multiple genes, the smallest p-value of genes is used.
  
- Open Targets Genetics (OTG):

  - Over 5000 genes are associated with CD, but many have low Association Scores.
  
  - A threshold of 0.3 was applied, selecting 159 Open Target genes.
  
  - Gene names are colored blue.

- WES Genes:

  - Gene names are colored red.

  - Genes present in both Open Targets and WES datasets are colored purple.
  
The following are annotated network from perturb-seq data:

![](assets/network_perturb_annotated.png){width="100%"}

Key findings:

A lot of hub are not GWAS significant genes, while peripheral genes are more likely GWAS significant genes. An explanation for this findings is that core genes tend not to be common variants, having more selection pressure. 


## Week 9-10: Mar 3 - Mar 16

One of the goal of this project is really to interpret and explain the GWAS findings of IBD. A simple way to do this is to summarize how many GWAS significant loci are core or peripheral genes. We summarize COTA result as the following:

**COTA Genes Detected**

|                            | COTA peripheral genes | COTA master peripheral genes | COTA core genes | Total number of genes explained in COTA network (peripheral + core) |
|---------------------------:|---------------------:|----------------------------:|----------------:|--------------------------------------------------------------------:|
| **SNP Based**              | 5                    | 7                           | 5               | 17                                                                 |
| **Gene Based**             | 7                    | 0                           | 7               | 14                                                                 |
| **Perturb Seq**            | 64                   | 18                          | 24              | 106                                                                |
| **Three Analysis (Unique Genes)** | 76                   | 25                          | 31 (4 genes overlapped in three analysis) | 132                                                                |

- Number of peripheral genes (regulate 1 core gene), number of master peripheral genes (regulate > 1 core genes), and number of core genes by using GWAS nearest genes

**GWAS Nearest Genes**

|                          | COTA peripheral genes | COTA master peripheral genes | COTA core genes | Total number of genes explained in COTA network (peripheral + core) |
|-------------------------:|---------------------:|----------------------------:|----------------:|--------------------------------------------------------------------:|
| **SNP Based**            | 1                    | 6                           | 3               | 10                                                                 |
| **Gene Based**           | 2                    | 0                           | 2               | 4                                                                  |
| **Perturb Seq**          | 14                   | 1                           | 4               | 19                                                                 |
| **Three Analysis (Unique Genes)** | 17                   | 7                           | 7 (2 genes overlapped in three analysis) | 31                                                                |


- Number of peripheral genes (regulate 1 core gene), number of master peripheral genes (regulate > 1 core genes), and number of core genes by using 1MB of the GWAS loci

**1Mb GWAS Significant Loci**

|                          | COTA peripheral genes | COTA master peripheral genes | COTA core genes | Total number of genes explained in COTA network (peripheral + core) |
|-------------------------:|---------------------:|----------------------------:|----------------:|--------------------------------------------------------------------:|
| **SNP Based**            | 4                    | 6                           | 3               | 13                                                                 |
| **Gene Based**           | 4                    | 0                           | 3               | 7                                                                  |
| **Perturb Seq**          | 31                   | 3                           | 6               | 40                                                                 |
| **Three Analysis (Unique Genes)** | 39                   | 9                           | 9 (2 genes overlapped in three analysis) | 57                                                                |


## Sept: Analysis on relevance of K562 to IBD

One of the issue of K562 perturb sep data is how relevant it is to IBD since K562 are red cell blood cell line. Couple of analysis are done to assess the relavence of K562 on IBD. 

#### Cell type annotation of K562 data

One way is to annotate each cell in K562 data. The UMAP of K562 expression data looks like the following:


![](assets/k562_expr_umap.png){width="100%"}

From the UMAP, it seems all expression are clustered together. There is no obvious clusters from UMAP, this may suggest K562 cells are more likely to be genetically identical. 

To be more accurate on cell type annotation, we use scmap (R package) to annotate K562 data based on a reference data ([1k1k](https://cellxgene.cziscience.com/collections/dde06e0f-ab3b-46be-96a2-a8082383c4a1)), which is a single cell data for autoimmune disease. The overall procedure of ***scmapCell*** based on comparing similarities (i.e. difference of experession of each cell to mean expression of each subclusters) of projection cells to subclusters analyzed from reference data. 

- In progress: require a lot memory and time.

#### P-value trend comparison of K562 data

Another analysis is to compare the pvalue from K562 perturb seq analysis to eQTL (whole blood) data and Jurkat T perturb seq data. This is to check whether trans pvalue have similar trend. 

- K562 vs. DGN (whole blood eQTL)

![](assets/dgn_k562_pvalue.png){width="100%"}


- K562 vs. Jurkat T (human T-lymphocyte cell line derived from T-cell leukemia)

![](assets/jurkat_k562_pvalue.png){width="100%"}

Jurkat T (human T-lymphocyte cell line derived from T-cell leukemia) might be more relavent to IBD.          

Compared to DGN data, there are more pvalues lines in the dotted line, that pvalue for two datasets are the same. However the range of pvalue are quite different for Jurkat and K562, which might due to sample size. 


The trend of pvalue in Jurkat and K562 are different. 

## Oct: Analysis on other Perturb-seq data

### Nadig Jurkat T cell perturb seq

Nadig Jurkat T cell:

- 260487 cells
- 8588 target genes, essential genes
- 2267 guides


QC steps: 

- Remove cells without sgRNA
- Keep sgRNA carried by more than 20 cells
- Remove cells with mito percent > 20
- Remove chrX/Y, chrM

Visulization of results

- Negative Control of pvalue

![](assets/nadig_sceptre_negative_control.png){width="100%"}


- QQ-Plot of pvalue

![](assets/qqplot_nadig_jurkat_essential_genes.png){width="100%"}

- Histogram of pvalue 

![](assets/nadig_sceptre_hist_pval.png){width="100%"}

- Number of pvalue under threshold

| threshold  | count     | proportion |
|------------|----------:|-----------:|
| 0.05       | 2,769,547 | 0.155     |
| 0.01       | 1,434,103 | 0.0804    |
| 0.001      |   729,389 | 0.0409    |
| 0.0001     |   444,594 | 0.0249    |
| 0.00001    |   299,523 | 0.0168    |


### Dandelion on Nadig Jurkat T perturb seq

Fisrt, check the overlap of burdent test genes and response genes from perturb seq

![](assets/burden_test_perturb_target_overlap.png){width="100%"}

Then, performing Dandelion on trans effect between genes by using Sceptre

- Peripheral genes overlapped with GWAS nearest genes as candidate peripheral genes

![](assets/network_annotated_gwas_overlap_jurkat_essential_genes.png){width="100%"}
  - **Disease Proximal Genes (DPGs)**: IL10RB, DNMT3A, ATG4C, TSPAN17, SF3B1, STRADB, USP36, LCP2
  - ATG4C, and DNMT3A are burdent test significant genes
  - LCP2, ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes, regulating by multiple peripheral genes


- All peripheral genes as candidate peripheral genes

![](assets/network_annotated_all_jurkat_essential_genes.png){width="100%"}


  - **Disease Proximal Genes (DPGs)**: IL10RB, DNMT3A, ATG4C, TSPAN17, SF3B1, STRADB, USP36, LCP2, GNAI3, ING1
  - ATG4C, and DNMT3A are burdent test significant genes
  - LCP2, ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes, regulating by multiple peripheral genes
  
  
#### Repeat all Dandelion analysis on it with mito percent 

Fisrt, the correlation between 2 result

![](correlation_sceptre_mito_percent.png){width="100%"}

Then, performing Dandelion on trans effect between genes by using Sceptre

- Peripheral genes overlapped with GWAS nearest genes as candidate peripheral genes

![](assets/network_annotated_gwas_overlap_jurkat_essential_genes_mito_percent.pdf){width="100%"}
  - **Disease Proximal Genes (DPGs)**: IL10RB, DNMT3A, ATG4C, TSPAN17, SF3B1, STRADB, USP36, LCP2
  - ATG4C, and DNMT3A are burdent test significant genes
  - LCP2, ATG4C, USP36, IL10RB, TSPAN17, STRADB are hub genes, regulating by multiple peripheral genes


- All peripheral genes as candidate peripheral genes

![](assets/network_annotated_all_jurkat_essential_genes_mito_percent.pdf){width="100%"}
  
 - GNAI3 is missing from this
  
  
### Cross-mappability analysis

[Cross Mappability](https://pmc.ncbi.nlm.nih.gov/articles/PMC6305209/) for gene A to gene B (crossmap(A,B), defined as the number of gene A's k-mers (75-mers from exons and 36-mers from UTRs) whose alignment start within exonic or untranslated regions of gene B. Only genes with mappability < 1 (not NA) appear in cross mapping files. In other words, it is the count of how many k-mers from A map to B. 

Because crossmap(A, B) $\ne$ crossmap(B, A) (A may have longer exons or more repetitive sequence), a symmetric version is defined $\frac{\text{crossmap(A,B)} + \text{crossmap(B,A)}}{2}$.

The range for cross mappability is larger than 0. And the summary for cross mappability as the following

- Distribution of cross mappability

![](assets/hist_cross_mappability.png){width="100%"}

![](assets/box_plot_cross_mappability.png){width="100%"}

| Percentile | Cross Mappability |
|:-----------:|:----------------:|
| 0%          | 1                |
| 10%         | 2                |
| 20%         | 4                |
| 30%         | 7                |
| 40%         | 10               |
| 50%         | 14               |
| 60%         | 19               |
| 70%         | 26               |
| 80%         | 36               |
| 90%         | 58               |
| 100%        | 16921            |




Most of the cross-mappability is less than 30. It is heavy tail, a lot of outliers. 

- Distribution of symmetric cross mappability

![](assets/hist_symmetric_cross_mappability.png){width="100%"}

![](assets/box_plot_symmetric_cross_mappability.png){width="100%"}

| Percentile | Symmetric Cross Mappability |
|:-----------:|:---------------------------:|
| 0%          | 0.5                         |
| 10%         | 1.0                         |
| 20%         | 2.5                         |
| 30%         | 4.0                         |
| 40%         | 6.0                         |
| 50%         | 8.0                         |
| 60%         | 11.5                        |
| 70%         | 16.0                        |
| 80%         | 23.0                        |
| 90%         | 38.5                        |
| 100%        | 15660.5                     |


Similarly, most of the cross-mappability is less than 30. It is heavy tail, a lot of outliers. 


- Dicussion: how to set threshold to define high cross mappability

  - Normalization: require length of gene A ?
  - By distribution ?



## Q&A on the Project

1. Why just burden test

- Can be flexible to many gene effect methods

2. Why not leverage cis effect

- Cis effect may not be detected due to stronger selective constraints. 

### Next Steps
1. **Map GWAS Loci**
   - Utilize **SNPeff** and **Fine-mapping**
   - Apply **ABC method** ([Nature Link](https://www.nature.com/articles/s41586-024-07022-x))
2. **Perform Perturb-seq for trans-effects**
   - Utilize **CRISPRi**
   - Direct application in **COTA analysis**.


## Project Overview
1. Fine-mapping of GWAS loci

  - Find casual gene for each loci
  
3. Connect core gene to the network
  - Gene expressed in certain cell type
  - trans-regulation path also expressed in certain cell type
  
4. Interpret GWAS loci not related to the core genes
  - pathway
  
3. Next level: Difference in UC and CD
  - Shared 
  - Unique



