---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```



## Goal

- Investigate potential mechanisms that may underlie genetic variants that affect a gene's protein expression level in trans, but not its mRNA expression level. 

- Curate databases that capture different mechanisms including annotated strict protein complex partners, many different methods that identify protein-protein interaction networks, known signaling pathway, and more. 

- Develop approaches that may help distinguish trans-pQTLs that are driven by cell-composition effects versus intra-cellular trans-pQTL effects

## Jun 26 Update

## Test whether pQTL enriched in protein pairs

### Dataset

- PPI 

  - Bioplex: ~ 2900 pairs after overlapping with pQTL data

  - Hippie: ~ 1346 pairs after overlapping

  - Corum: ~ 1270 pairs after overlapping

  - String: ~ 3391 pairs after overlapping
  
- Gene module

  - DNG gene module by trans-PCO: 
    
    - 166 gene modules
    
    - 12000+ genes in total
    
    - over 1,000,000 gene pairs 


- QTL summary statistics

  - UKB-PPP: plasma
  
  - DGN: whole blood


### Prodecure 

1. For each gene/protein pair, pick up the SNP from the most significant transQTL (i.e. smallest pvalue) of gene/protein A.

2. For gene/protein B of this gene pair, pick up pvalue of QTL for this SNP and B. 

3. After fisrt 2 steps, there will be a pair of pvalues, then calculate the proportion that both pvalues are significant under certain pvalue threshold, i.e. the probability of there is transQTL for B given there is transQTL for A. 

4. Use bootstrap to somehow mimic the distribution of the probability. For each dataset, each time select certain number of pairs and calculate probability of this dataset. From boostrap, there will be distribution of probability, able to estimate standard errors and construct confidence intervals. 


Note: Not sure about which pvalue threshold and sample size of bootstrap, so have results for multiple pvalue thresholds and boostrap sample sizes. 



### Preliminary result

- Plot for probability of there is transQTL for B given there is transQTL for A with bootstrap

![](assets/ppi_prob_bootstrap.png){width="100%"}

The overall trend as expected, PPI pairs have higher probability then random pairs. And the error bar decrease as sample size of bootstrap increase. 

- Table for cisQTL for B given there is transQTL for A



```{r}
library(knitr)
library(kableExtra)

df <- data.frame(
  Dataset = c("bioplex", "corum", "hippie", "string", "random"),
  Total = c(5800, 2542, 2692, 6698, 4471),
  `trans-cis` = c(21, 25, 11, 63, 6)
)

kable(df, align = "c", caption = "Table: Total Number of Cis-Trans Pairs") %>%
  kable_styling(full_width = F, position = "center") %>%
  column_spec(1, width = "5cm") %>%
  column_spec(2, width = "4cm") %>%
  column_spec(3, width = "4cm")
```

Note: Total pairs is the number of unique pairs * 2

### Consideration & Disscussion

1. The tissue difference may have impact on the result

  - Different tissue of PPI datasets:, e.g whole blood cells for bioplex and general cells for others
  
  - Different tissue in pQTL and eQTL data
  
2. The technology difference may also have some impact 

  - e.g. bioplex test for interaction instead of complex

3. The difference in number of pairs


## July 31 - Update

- Plot for probability of there is transQTL for B given there is 
  transQTL for A with bootstrap - Updated

![](assets/ppi_prob_bootstrap-updated.png){width="100%"}


### Exclude SNPs colocalized with cell type composition

- Plot for probability of there is transQTL for B given there is 
  transQTL for A with bootstrap 
  
  - Exclude SNPs colocalized with cell type composition

![](assets/ppi_prob_bootstrap_exclude_cell_type_coloc.png){width="100%"}

The average proportion are relatively lower than before. But the trend remain the same.


### Test how many unsignificant snp for protein A is significant for protein B

- Plot for probability of there is transQTL for B given there is 
  non-transQTL for A with bootstrap 
  
- Same procedure as before but first extract snp that not significant for protein A

![](assets/ppi_prob_bootstrap_non_sig.png){width="100%"}

From the plot, the proportion of transQTL for B given non-transQTL for A is close to corresponding threshold. 

### Analysis on top 2 significant snps

This is to check whether most significant SNP for A is less than the cutoff when second significant SNP for A is less than the cutoff

1. For each pair, choose the top 2 significant SNP for A


2. check whether the SNP 2 (second significant snp) is less than cutoff for B


3. Check whether SNP 1 is less than cutoff when SNP 2 is less than cutoff or not


```{r}
library(knitr)
library(kableExtra)
library(dplyr)

# Define the tables
# Define all five matrices
bioplex <- matrix(c(1507, 192, 225, 3874), nrow = 2, byrow = TRUE,
                  dimnames = list(c("snp2 significant", "snp2 not significant"),
                                  c("snp1 significant", "snp1 not significant")))

corum <- matrix(c(813, 116, 126, 1480), nrow = 2, byrow = TRUE,
                dimnames = list(c("snp2 significant", "snp2 not significant"),
                                c("snp1 significant", "snp1 not significant")))

hippie <- matrix(c(853, 81, 118, 1640), nrow = 2, byrow = TRUE,
                 dimnames = list(c("snp2 significant", "snp2 not significant"),
                                 c("snp1 significant", "snp1 not significant")))

string <- matrix(c(2160, 217, 324, 4060), nrow = 2, byrow = TRUE,
                 dimnames = list(c("snp2 significant", "snp2 not significant"),
                                 c("snp1 significant", "snp1 not significant")))

random <- matrix(c(907, 185, 212, 3165), nrow = 2, byrow = TRUE,
                 dimnames = list(c("snp2 significant", "snp2 not significant"),
                                 c("snp1 significant", "snp1 not significant")))

format_matrix_with_proportion <- function(mat) {
  row_totals <- rowSums(mat)
  proportions <- sweep(mat, 1, row_totals, FUN = "/")
  
  # Format each cell as "count (proportion)"
  formatted <- matrix(
    paste0(mat, " (", sprintf("%.2f", proportions), ")"),
    nrow = nrow(mat),
    dimnames = dimnames(mat)
  )
  
  # Add total column with count only
  total_col <- paste0(row_totals)
  
  cbind(formatted, Total = total_col)
}

bioplex_with_props <- format_matrix_with_proportion(bioplex)
corum_with_props <- format_matrix_with_proportion(corum)
hippie_with_props <- format_matrix_with_proportion(hippie)
string_with_props <- format_matrix_with_proportion(string)
random_with_props <- format_matrix_with_proportion(random)

# Create and style each table
kable(bioplex_with_props, caption = "bioplex 0.05") %>%
  kable_styling(full_width = FALSE, position = "center")

kable(corum_with_props, caption = "corum 0.05") %>%
  kable_styling(full_width = FALSE, position = "center")

kable(hippie_with_props, caption = "hippie 0.05") %>%
  kable_styling(full_width = FALSE, position = "center")

kable(string_with_props, caption = "string 0.05") %>%
  kable_styling(full_width = FALSE, position = "center")

kable(random_with_props, caption = "random 0.05") %>%
  kable_styling(full_width = FALSE, position = "center")

```

Based on the result, simple chi-square test is performed. Chi-square tests are significant, which means that whether SNP1 is significant is associated with whether SNP2 is significant or not. 

### Analysis on gene pairs

- Plot for probability of there is transQTL for B given there is 
  transQTL for A with bootstrap 
  
- Same procedure as before

![](assets/ppi_prob_bootstrap_with_dgn.png){width="100%"}



### Next step

1. Same thing in eQTL and CSF pQTL


2. Any other PPI module
 
 - [Production of Protein-Complex Components Is Stoichiometric and Lacks General Feedback Regulation in Eukaryotes](https://www.sciencedirect.com/science/article/pii/S2405471218304721): Protein complex from four species. 






